[2025-04-06T15:20:27.259+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: gcs_file_upload_dag.run_data_loading scheduled__2025-04-06T15:00:00+00:00 [queued]>
[2025-04-06T15:20:27.305+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: gcs_file_upload_dag.run_data_loading scheduled__2025-04-06T15:00:00+00:00 [queued]>
[2025-04-06T15:20:27.314+0000] {taskinstance.py:2170} INFO - Starting attempt 1 of 2
[2025-04-06T15:20:27.368+0000] {taskinstance.py:2191} INFO - Executing <Task(PythonOperator): run_data_loading> on 2025-04-06 15:00:00+00:00
[2025-04-06T15:20:27.426+0000] {standard_task_runner.py:60} INFO - Started process 654 to run task
[2025-04-06T15:20:27.435+0000] {standard_task_runner.py:87} INFO - Running: ['***', 'tasks', 'run', 'gcs_file_upload_dag', 'run_data_loading', 'scheduled__2025-04-06T15:00:00+00:00', '--job-id', '213', '--raw', '--subdir', 'DAGS_FOLDER/gcs_fileupload_dag.py', '--cfg-path', '/tmp/tmp4p3l98da']
[2025-04-06T15:20:27.465+0000] {standard_task_runner.py:88} INFO - Job 213: Subtask run_data_loading
[2025-04-06T15:20:27.513+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.8/site-packages/***/settings.py:194 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2025-04-06T15:20:27.638+0000] {task_command.py:423} INFO - Running <TaskInstance: gcs_file_upload_dag.run_data_loading scheduled__2025-04-06T15:00:00+00:00 [running]> on host 360c490b0eef
[2025-04-06T15:20:27.876+0000] {taskinstance.py:2480} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='gcs_file_upload_dag' AIRFLOW_CTX_TASK_ID='run_data_loading' AIRFLOW_CTX_EXECUTION_DATE='2025-04-06T15:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2025-04-06T15:00:00+00:00'
[2025-04-06T15:20:27.888+0000] {gcs_fileupload_dag.py:16} INFO - Calling data_loading from DAG...
[2025-04-06T15:20:27.898+0000] {logging_mixin.py:188} INFO - Starting data_loading...
[2025-04-06T15:20:28.247+0000] {logging_mixin.py:188} INFO - GIT API rate limit: 5000/5000
[2025-04-06T15:26:17.012+0000] {logging_mixin.py:188} WARNING - Processing Ai-app-by-using-python: 0it [00:00, ?it/s]
[2025-04-06T15:26:17.029+0000] {_metadata.py:139} WARNING - Compute Engine Metadata server unavailable on attempt 1 of 3. Reason: [Errno 111] Connection refused
[2025-04-06T15:26:17.036+0000] {_metadata.py:139} WARNING - Compute Engine Metadata server unavailable on attempt 2 of 3. Reason: [Errno 111] Connection refused
[2025-04-06T15:26:17.039+0000] {_metadata.py:139} WARNING - Compute Engine Metadata server unavailable on attempt 3 of 3. Reason: [Errno 111] Connection refused
[2025-04-06T15:26:17.040+0000] {_default.py:338} WARNING - Authentication failed using Compute Engine authentication due to unavailable metadata server.
[2025-04-06T15:26:17.042+0000] {logging_mixin.py:188} INFO - Error initializing GCS client: Your default credentials were not found. To set up Application Default Credentials, see https://cloud.google.com/docs/authentication/external/set-up-adc for more information.
[2025-04-06T15:26:17.042+0000] {logging_mixin.py:188} INFO - Deleted unneeded file: /tmp/Ai-app-by-using-python/.gitignore
[2025-04-06T15:26:17.049+0000] {_metadata.py:139} WARNING - Compute Engine Metadata server unavailable on attempt 1 of 3. Reason: [Errno 111] Connection refused
[2025-04-06T15:26:17.058+0000] {_metadata.py:139} WARNING - Compute Engine Metadata server unavailable on attempt 2 of 3. Reason: [Errno 111] Connection refused
[2025-04-06T15:26:17.063+0000] {_metadata.py:139} WARNING - Compute Engine Metadata server unavailable on attempt 3 of 3. Reason: [Errno 111] Connection refused
[2025-04-06T15:26:17.064+0000] {_default.py:338} WARNING - Authentication failed using Compute Engine authentication due to unavailable metadata server.
[2025-04-06T15:26:17.064+0000] {logging_mixin.py:188} INFO - Error initializing GCS client: Your default credentials were not found. To set up Application Default Credentials, see https://cloud.google.com/docs/authentication/external/set-up-adc for more information.
[2025-04-06T15:26:17.066+0000] {logging_mixin.py:188} INFO - Deleted unneeded file: /tmp/Ai-app-by-using-python/.git/HEAD
[2025-04-06T15:26:17.067+0000] {logging_mixin.py:188} INFO - Deleted unneeded file: /tmp/Ai-app-by-using-python/.git/index
[2025-04-06T15:26:17.068+0000] {logging_mixin.py:188} INFO - Deleted unneeded file: /tmp/Ai-app-by-using-python/.git/packed-refs
[2025-04-06T15:26:17.068+0000] {logging_mixin.py:188} INFO - Deleted unneeded file: /tmp/Ai-app-by-using-python/.git/description
[2025-04-06T15:26:17.069+0000] {logging_mixin.py:188} INFO - Deleted unneeded file: /tmp/Ai-app-by-using-python/.git/config
[2025-04-06T15:26:17.071+0000] {logging_mixin.py:188} INFO - Deleted unneeded file: /tmp/Ai-app-by-using-python/.git/refs/remotes/origin/HEAD
[2025-04-06T15:26:17.072+0000] {logging_mixin.py:188} INFO - Deleted unneeded file: /tmp/Ai-app-by-using-python/.git/refs/heads/main
[2025-04-06T15:26:17.079+0000] {logging_mixin.py:188} INFO - Deleted unneeded file: /tmp/Ai-app-by-using-python/.git/logs/HEAD
[2025-04-06T15:26:17.080+0000] {logging_mixin.py:188} INFO - Deleted unneeded file: /tmp/Ai-app-by-using-python/.git/logs/refs/remotes/origin/HEAD
[2025-04-06T15:26:17.081+0000] {logging_mixin.py:188} INFO - Deleted unneeded file: /tmp/Ai-app-by-using-python/.git/logs/refs/heads/main
[2025-04-06T15:26:17.082+0000] {logging_mixin.py:188} INFO - Deleted unneeded file: /tmp/Ai-app-by-using-python/.git/objects/pack/pack-b3efe16abf8595f1fe15217968f918b4f585d1a0.pack
[2025-04-06T15:26:17.083+0000] {logging_mixin.py:188} INFO - Deleted unneeded file: /tmp/Ai-app-by-using-python/.git/objects/pack/pack-b3efe16abf8595f1fe15217968f918b4f585d1a0.idx
[2025-04-06T15:26:17.084+0000] {logging_mixin.py:188} INFO - Deleted unneeded file: /tmp/Ai-app-by-using-python/.git/hooks/pre-receive.sample
[2025-04-06T15:26:17.085+0000] {logging_mixin.py:188} INFO - Deleted unneeded file: /tmp/Ai-app-by-using-python/.git/hooks/fsmonitor-watchman.sample
[2025-04-06T15:26:17.086+0000] {logging_mixin.py:188} INFO - Deleted unneeded file: /tmp/Ai-app-by-using-python/.git/hooks/push-to-checkout.sample
[2025-04-06T15:26:17.090+0000] {logging_mixin.py:188} INFO - Deleted unneeded file: /tmp/Ai-app-by-using-python/.git/hooks/commit-msg.sample
[2025-04-06T15:26:17.096+0000] {logging_mixin.py:188} INFO - Deleted unneeded file: /tmp/Ai-app-by-using-python/.git/hooks/pre-rebase.sample
[2025-04-06T15:26:17.099+0000] {logging_mixin.py:188} INFO - Deleted unneeded file: /tmp/Ai-app-by-using-python/.git/hooks/update.sample
[2025-04-06T15:26:17.101+0000] {logging_mixin.py:188} INFO - Deleted unneeded file: /tmp/Ai-app-by-using-python/.git/hooks/post-update.sample
[2025-04-06T15:26:17.102+0000] {logging_mixin.py:188} INFO - Deleted unneeded file: /tmp/Ai-app-by-using-python/.git/hooks/pre-merge-commit.sample
[2025-04-06T15:26:17.103+0000] {logging_mixin.py:188} INFO - Deleted unneeded file: /tmp/Ai-app-by-using-python/.git/hooks/prepare-commit-msg.sample
[2025-04-06T15:26:17.104+0000] {logging_mixin.py:188} INFO - Deleted unneeded file: /tmp/Ai-app-by-using-python/.git/hooks/pre-applypatch.sample
[2025-04-06T15:26:17.105+0000] {logging_mixin.py:188} INFO - Deleted unneeded file: /tmp/Ai-app-by-using-python/.git/hooks/applypatch-msg.sample
[2025-04-06T15:26:17.106+0000] {logging_mixin.py:188} INFO - Deleted unneeded file: /tmp/Ai-app-by-using-python/.git/hooks/pre-push.sample
[2025-04-06T15:26:17.109+0000] {logging_mixin.py:188} INFO - Deleted unneeded file: /tmp/Ai-app-by-using-python/.git/hooks/pre-commit.sample
[2025-04-06T15:26:17.110+0000] {logging_mixin.py:188} INFO - Deleted unneeded file: /tmp/Ai-app-by-using-python/.git/info/exclude
[2025-04-06T15:26:17.111+0000] {logging_mixin.py:188} WARNING - Processing Ai-app-by-using-python: 18it [00:00, 188.49it/s]
[2025-04-06T15:26:17.114+0000] {logging_mixin.py:188} INFO - Deleted temporary folder: /tmp/Ai-app-by-using-python
[2025-04-06T15:26:17.115+0000] {logging_mixin.py:188} INFO - Repository MaxBrowser already cloned. Skipping.
[2025-04-06T15:26:17.117+0000] {logging_mixin.py:188} INFO - Repository portfolio already cloned. Skipping.
[2025-04-06T15:26:17.918+0000] {logging_mixin.py:188} WARNING - Processing learning-python: 0it [00:00, ?it/s]
[2025-04-06T15:26:17.928+0000] {_metadata.py:139} WARNING - Compute Engine Metadata server unavailable on attempt 1 of 3. Reason: [Errno 111] Connection refused
[2025-04-06T15:26:17.939+0000] {_metadata.py:139} WARNING - Compute Engine Metadata server unavailable on attempt 2 of 3. Reason: [Errno 111] Connection refused
[2025-04-06T15:26:17.952+0000] {_metadata.py:139} WARNING - Compute Engine Metadata server unavailable on attempt 3 of 3. Reason: [Errno 111] Connection refused
[2025-04-06T15:26:17.954+0000] {_default.py:338} WARNING - Authentication failed using Compute Engine authentication due to unavailable metadata server.
[2025-04-06T15:26:17.958+0000] {logging_mixin.py:188} INFO - Error initializing GCS client: Your default credentials were not found. To set up Application Default Credentials, see https://cloud.google.com/docs/authentication/external/set-up-adc for more information.
[2025-04-06T15:26:17.961+0000] {logging_mixin.py:188} INFO - Deleted unneeded file: /tmp/learning-python/README.md
[2025-04-06T15:26:17.964+0000] {logging_mixin.py:188} INFO - Deleted unneeded file: /tmp/learning-python/s
[2025-04-06T15:26:17.973+0000] {_metadata.py:139} WARNING - Compute Engine Metadata server unavailable on attempt 1 of 3. Reason: [Errno 111] Connection refused
[2025-04-06T15:26:17.987+0000] {_metadata.py:139} WARNING - Compute Engine Metadata server unavailable on attempt 2 of 3. Reason: [Errno 111] Connection refused
[2025-04-06T15:26:18.017+0000] {_metadata.py:139} WARNING - Compute Engine Metadata server unavailable on attempt 3 of 3. Reason: [Errno 111] Connection refused
[2025-04-06T15:26:18.023+0000] {_default.py:338} WARNING - Authentication failed using Compute Engine authentication due to unavailable metadata server.
[2025-04-06T15:26:18.029+0000] {logging_mixin.py:188} INFO - Error initializing GCS client: Your default credentials were not found. To set up Application Default Credentials, see https://cloud.google.com/docs/authentication/external/set-up-adc for more information.
[2025-04-06T15:26:18.035+0000] {logging_mixin.py:188} WARNING - Processing learning-python: 1it [00:00,  8.86it/s]
[2025-04-06T15:26:18.044+0000] {logging_mixin.py:188} INFO - Deleted unneeded file: /tmp/learning-python/.git/HEAD
[2025-04-06T15:26:18.060+0000] {logging_mixin.py:188} INFO - Deleted unneeded file: /tmp/learning-python/.git/index
[2025-04-06T15:26:18.073+0000] {logging_mixin.py:188} INFO - Deleted unneeded file: /tmp/learning-python/.git/packed-refs
[2025-04-06T15:26:18.081+0000] {logging_mixin.py:188} INFO - Deleted unneeded file: /tmp/learning-python/.git/description
[2025-04-06T15:26:18.085+0000] {logging_mixin.py:188} INFO - Deleted unneeded file: /tmp/learning-python/.git/config
[2025-04-06T15:26:18.092+0000] {logging_mixin.py:188} INFO - Deleted unneeded file: /tmp/learning-python/.git/refs/remotes/origin/HEAD
[2025-04-06T15:26:18.101+0000] {logging_mixin.py:188} INFO - Deleted unneeded file: /tmp/learning-python/.git/refs/heads/main
[2025-04-06T15:26:18.108+0000] {logging_mixin.py:188} INFO - Deleted unneeded file: /tmp/learning-python/.git/logs/HEAD
[2025-04-06T15:26:18.120+0000] {logging_mixin.py:188} INFO - Deleted unneeded file: /tmp/learning-python/.git/logs/refs/remotes/origin/HEAD
[2025-04-06T15:26:18.127+0000] {logging_mixin.py:188} INFO - Deleted unneeded file: /tmp/learning-python/.git/logs/refs/heads/main
[2025-04-06T15:26:18.136+0000] {logging_mixin.py:188} WARNING - Processing learning-python: 12it [00:00, 64.90it/s]
[2025-04-06T15:26:18.146+0000] {logging_mixin.py:188} INFO - Deleted unneeded file: /tmp/learning-python/.git/objects/pack/pack-b99511549b941377f214af430801c9871a213a57.idx
[2025-04-06T15:26:18.154+0000] {logging_mixin.py:188} INFO - Deleted unneeded file: /tmp/learning-python/.git/objects/pack/pack-b99511549b941377f214af430801c9871a213a57.pack
[2025-04-06T15:26:18.163+0000] {logging_mixin.py:188} INFO - Deleted unneeded file: /tmp/learning-python/.git/hooks/pre-receive.sample
[2025-04-06T15:26:18.171+0000] {logging_mixin.py:188} INFO - Deleted unneeded file: /tmp/learning-python/.git/hooks/fsmonitor-watchman.sample
[2025-04-06T15:26:18.176+0000] {logging_mixin.py:188} INFO - Deleted unneeded file: /tmp/learning-python/.git/hooks/push-to-checkout.sample
[2025-04-06T15:26:18.180+0000] {logging_mixin.py:188} INFO - Deleted unneeded file: /tmp/learning-python/.git/hooks/commit-msg.sample
[2025-04-06T15:26:18.187+0000] {logging_mixin.py:188} INFO - Deleted unneeded file: /tmp/learning-python/.git/hooks/pre-rebase.sample
[2025-04-06T15:26:18.193+0000] {logging_mixin.py:188} INFO - Deleted unneeded file: /tmp/learning-python/.git/hooks/update.sample
[2025-04-06T15:26:18.199+0000] {logging_mixin.py:188} INFO - Deleted unneeded file: /tmp/learning-python/.git/hooks/post-update.sample
[2025-04-06T15:26:18.207+0000] {logging_mixin.py:188} INFO - Deleted unneeded file: /tmp/learning-python/.git/hooks/pre-merge-commit.sample
[2025-04-06T15:26:18.213+0000] {logging_mixin.py:188} INFO - Deleted unneeded file: /tmp/learning-python/.git/hooks/prepare-commit-msg.sample
[2025-04-06T15:26:18.216+0000] {logging_mixin.py:188} INFO - Deleted unneeded file: /tmp/learning-python/.git/hooks/pre-applypatch.sample
[2025-04-06T15:26:18.223+0000] {logging_mixin.py:188} INFO - Deleted unneeded file: /tmp/learning-python/.git/hooks/applypatch-msg.sample
[2025-04-06T15:26:18.229+0000] {logging_mixin.py:188} INFO - Deleted unneeded file: /tmp/learning-python/.git/hooks/pre-push.sample
[2025-04-06T15:26:18.233+0000] {logging_mixin.py:188} INFO - Deleted unneeded file: /tmp/learning-python/.git/hooks/pre-commit.sample
[2025-04-06T15:26:18.237+0000] {logging_mixin.py:188} INFO - Deleted unneeded file: /tmp/learning-python/.git/info/exclude
[2025-04-06T15:26:18.243+0000] {logging_mixin.py:188} WARNING - Processing learning-python: 18it [00:00, 56.05it/s]
[2025-04-06T15:26:18.251+0000] {logging_mixin.py:188} INFO - Deleted temporary folder: /tmp/learning-python
[2025-04-06T15:26:18.256+0000] {logging_mixin.py:188} INFO - Repository ToDo-List already cloned. Skipping.
[2025-04-06T15:26:24.009+0000] {logging_mixin.py:188} WARNING - Processing Real_Time_Voice_Translator_Web: 0it [00:00, ?it/s]
[2025-04-06T15:26:24.013+0000] {logging_mixin.py:188} INFO - Deleted unneeded file: /tmp/Real_Time_Voice_Translator_Web/README.md
[2025-04-06T15:26:24.015+0000] {logging_mixin.py:188} INFO - Deleted unneeded file: /tmp/Real_Time_Voice_Translator_Web/.git/HEAD
[2025-04-06T15:26:24.017+0000] {logging_mixin.py:188} INFO - Deleted unneeded file: /tmp/Real_Time_Voice_Translator_Web/.git/index
[2025-04-06T15:26:24.019+0000] {logging_mixin.py:188} INFO - Deleted unneeded file: /tmp/Real_Time_Voice_Translator_Web/.git/packed-refs
[2025-04-06T15:26:24.020+0000] {logging_mixin.py:188} INFO - Deleted unneeded file: /tmp/Real_Time_Voice_Translator_Web/.git/description
[2025-04-06T15:26:24.022+0000] {logging_mixin.py:188} INFO - Deleted unneeded file: /tmp/Real_Time_Voice_Translator_Web/.git/config
[2025-04-06T15:26:24.024+0000] {logging_mixin.py:188} INFO - Deleted unneeded file: /tmp/Real_Time_Voice_Translator_Web/.git/refs/remotes/origin/HEAD
[2025-04-06T15:26:24.026+0000] {logging_mixin.py:188} INFO - Deleted unneeded file: /tmp/Real_Time_Voice_Translator_Web/.git/refs/heads/main
[2025-04-06T15:26:24.027+0000] {logging_mixin.py:188} INFO - Deleted unneeded file: /tmp/Real_Time_Voice_Translator_Web/.git/logs/HEAD
[2025-04-06T15:26:24.029+0000] {logging_mixin.py:188} INFO - Deleted unneeded file: /tmp/Real_Time_Voice_Translator_Web/.git/logs/refs/remotes/origin/HEAD
[2025-04-06T15:26:24.033+0000] {logging_mixin.py:188} INFO - Deleted unneeded file: /tmp/Real_Time_Voice_Translator_Web/.git/logs/refs/heads/main
[2025-04-06T15:26:24.034+0000] {logging_mixin.py:188} INFO - Deleted unneeded file: /tmp/Real_Time_Voice_Translator_Web/.git/objects/pack/pack-7f6dac6d3e6fb1c30e6715d92d9f9bd0a4f66f0f.idx
[2025-04-06T15:26:24.036+0000] {logging_mixin.py:188} INFO - Deleted unneeded file: /tmp/Real_Time_Voice_Translator_Web/.git/objects/pack/pack-7f6dac6d3e6fb1c30e6715d92d9f9bd0a4f66f0f.pack
[2025-04-06T15:26:24.037+0000] {logging_mixin.py:188} INFO - Deleted unneeded file: /tmp/Real_Time_Voice_Translator_Web/.git/hooks/pre-receive.sample
[2025-04-06T15:26:24.038+0000] {logging_mixin.py:188} INFO - Deleted unneeded file: /tmp/Real_Time_Voice_Translator_Web/.git/hooks/fsmonitor-watchman.sample
[2025-04-06T15:26:24.039+0000] {logging_mixin.py:188} INFO - Deleted unneeded file: /tmp/Real_Time_Voice_Translator_Web/.git/hooks/push-to-checkout.sample
[2025-04-06T15:26:24.040+0000] {logging_mixin.py:188} INFO - Deleted unneeded file: /tmp/Real_Time_Voice_Translator_Web/.git/hooks/commit-msg.sample
[2025-04-06T15:26:24.045+0000] {logging_mixin.py:188} INFO - Deleted unneeded file: /tmp/Real_Time_Voice_Translator_Web/.git/hooks/pre-rebase.sample
[2025-04-06T15:26:24.047+0000] {logging_mixin.py:188} INFO - Deleted unneeded file: /tmp/Real_Time_Voice_Translator_Web/.git/hooks/update.sample
[2025-04-06T15:26:24.049+0000] {logging_mixin.py:188} INFO - Deleted unneeded file: /tmp/Real_Time_Voice_Translator_Web/.git/hooks/post-update.sample
[2025-04-06T15:26:24.049+0000] {logging_mixin.py:188} INFO - Deleted unneeded file: /tmp/Real_Time_Voice_Translator_Web/.git/hooks/pre-merge-commit.sample
[2025-04-06T15:26:24.050+0000] {logging_mixin.py:188} INFO - Deleted unneeded file: /tmp/Real_Time_Voice_Translator_Web/.git/hooks/prepare-commit-msg.sample
[2025-04-06T15:26:24.051+0000] {logging_mixin.py:188} INFO - Deleted unneeded file: /tmp/Real_Time_Voice_Translator_Web/.git/hooks/pre-applypatch.sample
[2025-04-06T15:26:24.052+0000] {logging_mixin.py:188} INFO - Deleted unneeded file: /tmp/Real_Time_Voice_Translator_Web/.git/hooks/applypatch-msg.sample
[2025-04-06T15:26:24.053+0000] {logging_mixin.py:188} INFO - Deleted unneeded file: /tmp/Real_Time_Voice_Translator_Web/.git/hooks/pre-push.sample
[2025-04-06T15:26:24.055+0000] {logging_mixin.py:188} INFO - Deleted unneeded file: /tmp/Real_Time_Voice_Translator_Web/.git/hooks/pre-commit.sample
[2025-04-06T15:26:24.056+0000] {logging_mixin.py:188} INFO - Deleted unneeded file: /tmp/Real_Time_Voice_Translator_Web/.git/info/exclude
[2025-04-06T15:26:24.063+0000] {logging_mixin.py:188} INFO - Deleted unneeded file: /tmp/Real_Time_Voice_Translator_Web/real_time_voice_translator_web/requirements.txt
[2025-04-06T15:26:24.072+0000] {_metadata.py:139} WARNING - Compute Engine Metadata server unavailable on attempt 1 of 3. Reason: [Errno 111] Connection refused
[2025-04-06T15:26:24.077+0000] {_metadata.py:139} WARNING - Compute Engine Metadata server unavailable on attempt 2 of 3. Reason: [Errno 111] Connection refused
[2025-04-06T15:26:24.081+0000] {_metadata.py:139} WARNING - Compute Engine Metadata server unavailable on attempt 3 of 3. Reason: [Errno 111] Connection refused
[2025-04-06T15:26:24.082+0000] {_default.py:338} WARNING - Authentication failed using Compute Engine authentication due to unavailable metadata server.
[2025-04-06T15:26:24.083+0000] {logging_mixin.py:188} INFO - Error initializing GCS client: Your default credentials were not found. To set up Application Default Credentials, see https://cloud.google.com/docs/authentication/external/set-up-adc for more information.
[2025-04-06T15:26:24.084+0000] {logging_mixin.py:188} INFO - Deleted unneeded file: /tmp/Real_Time_Voice_Translator_Web/real_time_voice_translator_web/static/voices/68fd4b78-0f63-4032-ba84-4d81f56fe04d.mp3
[2025-04-06T15:26:24.086+0000] {logging_mixin.py:188} INFO - Deleted unneeded file: /tmp/Real_Time_Voice_Translator_Web/real_time_voice_translator_web/static/voices/c0e10fce-853f-4317-98b5-5afb02d49f80.mp3
[2025-04-06T15:26:24.087+0000] {logging_mixin.py:188} INFO - Deleted unneeded file: /tmp/Real_Time_Voice_Translator_Web/real_time_voice_translator_web/static/voices/4c52bae9-ef22-470b-aa81-6a7b3664038c.mp3
[2025-04-06T15:26:24.087+0000] {logging_mixin.py:188} INFO - Deleted unneeded file: /tmp/Real_Time_Voice_Translator_Web/real_time_voice_translator_web/static/voices/c28a9f39-d480-4dc1-b1db-e89307da55e8.mp3
[2025-04-06T15:26:24.089+0000] {logging_mixin.py:188} INFO - Deleted unneeded file: /tmp/Real_Time_Voice_Translator_Web/real_time_voice_translator_web/static/voices/1f3504af-1c76-4dce-be7c-a731fae6adef.mp3
[2025-04-06T15:26:24.097+0000] {_metadata.py:139} WARNING - Compute Engine Metadata server unavailable on attempt 1 of 3. Reason: [Errno 111] Connection refused
[2025-04-06T15:26:24.102+0000] {_metadata.py:139} WARNING - Compute Engine Metadata server unavailable on attempt 2 of 3. Reason: [Errno 111] Connection refused
[2025-04-06T15:26:24.107+0000] {_metadata.py:139} WARNING - Compute Engine Metadata server unavailable on attempt 3 of 3. Reason: [Errno 111] Connection refused
[2025-04-06T15:26:24.109+0000] {_default.py:338} WARNING - Authentication failed using Compute Engine authentication due to unavailable metadata server.
[2025-04-06T15:26:24.110+0000] {logging_mixin.py:188} INFO - Error initializing GCS client: Your default credentials were not found. To set up Application Default Credentials, see https://cloud.google.com/docs/authentication/external/set-up-adc for more information.
[2025-04-06T15:26:24.112+0000] {logging_mixin.py:188} INFO - Deleted unneeded file: /tmp/Real_Time_Voice_Translator_Web/real_time_voice_translator_web/static/js/scripts.js
[2025-04-06T15:26:24.115+0000] {_metadata.py:139} WARNING - Compute Engine Metadata server unavailable on attempt 1 of 3. Reason: [Errno 111] Connection refused
[2025-04-06T15:26:24.121+0000] {_metadata.py:139} WARNING - Compute Engine Metadata server unavailable on attempt 2 of 3. Reason: [Errno 111] Connection refused
[2025-04-06T15:26:24.125+0000] {_metadata.py:139} WARNING - Compute Engine Metadata server unavailable on attempt 3 of 3. Reason: [Errno 111] Connection refused
[2025-04-06T15:26:24.126+0000] {_default.py:338} WARNING - Authentication failed using Compute Engine authentication due to unavailable metadata server.
[2025-04-06T15:26:24.128+0000] {logging_mixin.py:188} INFO - Error initializing GCS client: Your default credentials were not found. To set up Application Default Credentials, see https://cloud.google.com/docs/authentication/external/set-up-adc for more information.
[2025-04-06T15:26:24.129+0000] {logging_mixin.py:188} WARNING - Processing Real_Time_Voice_Translator_Web: 24it [00:00, 207.03it/s]
[2025-04-06T15:26:24.130+0000] {logging_mixin.py:188} WARNING - Processing Real_Time_Voice_Translator_Web: 24it [00:00, 204.99it/s]
[2025-04-06T15:26:24.133+0000] {logging_mixin.py:188} INFO - Deleted temporary folder: /tmp/Real_Time_Voice_Translator_Web
[2025-04-06T15:26:24.134+0000] {logging_mixin.py:188} INFO - Repository text_classifcation_with_attention already cloned. Skipping.
[2025-04-06T15:26:25.880+0000] {logging_mixin.py:188} WARNING - Processing voiceover: 0it [00:00, ?it/s]
[2025-04-06T15:26:25.886+0000] {_metadata.py:139} WARNING - Compute Engine Metadata server unavailable on attempt 1 of 3. Reason: [Errno 111] Connection refused
[2025-04-06T15:26:25.891+0000] {_metadata.py:139} WARNING - Compute Engine Metadata server unavailable on attempt 2 of 3. Reason: [Errno 111] Connection refused
[2025-04-06T15:26:25.895+0000] {_metadata.py:139} WARNING - Compute Engine Metadata server unavailable on attempt 3 of 3. Reason: [Errno 111] Connection refused
[2025-04-06T15:26:25.896+0000] {_default.py:338} WARNING - Authentication failed using Compute Engine authentication due to unavailable metadata server.
[2025-04-06T15:26:25.897+0000] {logging_mixin.py:188} INFO - Error initializing GCS client: Your default credentials were not found. To set up Application Default Credentials, see https://cloud.google.com/docs/authentication/external/set-up-adc for more information.
[2025-04-06T15:26:25.898+0000] {logging_mixin.py:188} INFO - Deleted unneeded file: /tmp/voiceover/preload.js
[2025-04-06T15:26:25.899+0000] {logging_mixin.py:188} INFO - Deleted unneeded file: /tmp/voiceover/VideoCreator.js
[2025-04-06T15:26:25.899+0000] {logging_mixin.py:188} INFO - Deleted unneeded file: /tmp/voiceover/readme.md
[2025-04-06T15:26:25.900+0000] {logging_mixin.py:188} INFO - Deleted unneeded file: /tmp/voiceover/package.json
[2025-04-06T15:26:25.901+0000] {logging_mixin.py:188} INFO - Deleted unneeded file: /tmp/voiceover/requirements.txt
[2025-04-06T15:26:25.902+0000] {logging_mixin.py:188} INFO - Deleted unneeded file: /tmp/voiceover/.gitignore
[2025-04-06T15:26:25.903+0000] {logging_mixin.py:188} INFO - Deleted unneeded file: /tmp/voiceover/generate_audio.spec
[2025-04-06T15:26:25.903+0000] {logging_mixin.py:188} INFO - Deleted unneeded file: /tmp/voiceover/renderer.js
[2025-04-06T15:26:25.904+0000] {logging_mixin.py:188} INFO - Deleted unneeded file: /tmp/voiceover/package-lock.json
[2025-04-06T15:26:25.905+0000] {logging_mixin.py:188} INFO - Deleted unneeded file: /tmp/voiceover/.DS_Store
[2025-04-06T15:26:25.905+0000] {logging_mixin.py:188} INFO - Deleted unneeded file: /tmp/voiceover/forge.config.js
[2025-04-06T15:26:25.906+0000] {logging_mixin.py:188} INFO - Deleted unneeded file: /tmp/voiceover/temp_audio_with_silence.mp3
[2025-04-06T15:26:25.907+0000] {logging_mixin.py:188} INFO - Deleted unneeded file: /tmp/voiceover/main.js
[2025-04-06T15:26:25.910+0000] {_metadata.py:139} WARNING - Compute Engine Metadata server unavailable on attempt 1 of 3. Reason: [Errno 111] Connection refused
[2025-04-06T15:26:25.914+0000] {_metadata.py:139} WARNING - Compute Engine Metadata server unavailable on attempt 2 of 3. Reason: [Errno 111] Connection refused
[2025-04-06T15:26:25.919+0000] {_metadata.py:139} WARNING - Compute Engine Metadata server unavailable on attempt 3 of 3. Reason: [Errno 111] Connection refused
[2025-04-06T15:26:25.920+0000] {_default.py:338} WARNING - Authentication failed using Compute Engine authentication due to unavailable metadata server.
[2025-04-06T15:26:25.920+0000] {logging_mixin.py:188} INFO - Error initializing GCS client: Your default credentials were not found. To set up Application Default Credentials, see https://cloud.google.com/docs/authentication/external/set-up-adc for more information.
[2025-04-06T15:26:25.921+0000] {logging_mixin.py:188} INFO - Deleted unneeded file: /tmp/voiceover/.vscode/launch.json
[2025-04-06T15:26:25.922+0000] {logging_mixin.py:188} INFO - Deleted unneeded file: /tmp/voiceover/images/bg1.jpg
[2025-04-06T15:26:25.924+0000] {logging_mixin.py:188} INFO - Deleted unneeded file: /tmp/voiceover/fonts/Noto Sans Myanmar Regular.ttf
[2025-04-06T15:26:25.926+0000] {logging_mixin.py:188} INFO - Deleted unneeded file: /tmp/voiceover/fonts/Pyidaungsu-Bold.ttf
[2025-04-06T15:26:25.927+0000] {logging_mixin.py:188} INFO - Deleted unneeded file: /tmp/voiceover/fonts/MM3 Font.ttf
[2025-04-06T15:26:25.928+0000] {logging_mixin.py:188} INFO - Deleted unneeded file: /tmp/voiceover/output/transcript.srt
[2025-04-06T15:26:25.929+0000] {logging_mixin.py:188} INFO - Deleted unneeded file: /tmp/voiceover/output/transcript.txt
[2025-04-06T15:26:25.930+0000] {logging_mixin.py:188} INFO - Deleted unneeded file: /tmp/voiceover/output/video.mp4
[2025-04-06T15:26:25.931+0000] {logging_mixin.py:188} INFO - Deleted unneeded file: /tmp/voiceover/output/generated_audio.mp3
[2025-04-06T15:26:25.932+0000] {logging_mixin.py:188} INFO - Deleted unneeded file: /tmp/voiceover/.git/HEAD
[2025-04-06T15:26:25.933+0000] {logging_mixin.py:188} INFO - Deleted unneeded file: /tmp/voiceover/.git/index
[2025-04-06T15:26:25.934+0000] {logging_mixin.py:188} INFO - Deleted unneeded file: /tmp/voiceover/.git/packed-refs
[2025-04-06T15:26:25.935+0000] {logging_mixin.py:188} INFO - Deleted unneeded file: /tmp/voiceover/.git/description
[2025-04-06T15:26:25.937+0000] {logging_mixin.py:188} INFO - Deleted unneeded file: /tmp/voiceover/.git/config
[2025-04-06T15:26:25.938+0000] {logging_mixin.py:188} INFO - Deleted unneeded file: /tmp/voiceover/.git/refs/remotes/origin/HEAD
[2025-04-06T15:26:25.939+0000] {logging_mixin.py:188} INFO - Deleted unneeded file: /tmp/voiceover/.git/refs/heads/main
[2025-04-06T15:26:25.940+0000] {logging_mixin.py:188} INFO - Deleted unneeded file: /tmp/voiceover/.git/logs/HEAD
[2025-04-06T15:26:25.941+0000] {logging_mixin.py:188} INFO - Deleted unneeded file: /tmp/voiceover/.git/logs/refs/remotes/origin/HEAD
[2025-04-06T15:26:25.942+0000] {logging_mixin.py:188} INFO - Deleted unneeded file: /tmp/voiceover/.git/logs/refs/heads/main
[2025-04-06T15:26:25.943+0000] {logging_mixin.py:188} INFO - Deleted unneeded file: /tmp/voiceover/.git/objects/pack/pack-327234b18d1e8595eccc77cfb4ca998c25f8f4fa.idx
[2025-04-06T15:26:25.947+0000] {logging_mixin.py:188} INFO - Deleted unneeded file: /tmp/voiceover/.git/objects/pack/pack-327234b18d1e8595eccc77cfb4ca998c25f8f4fa.pack
[2025-04-06T15:26:25.949+0000] {logging_mixin.py:188} INFO - Deleted unneeded file: /tmp/voiceover/.git/hooks/pre-receive.sample
[2025-04-06T15:26:25.950+0000] {logging_mixin.py:188} INFO - Deleted unneeded file: /tmp/voiceover/.git/hooks/fsmonitor-watchman.sample
[2025-04-06T15:26:25.952+0000] {logging_mixin.py:188} INFO - Deleted unneeded file: /tmp/voiceover/.git/hooks/push-to-checkout.sample
[2025-04-06T15:26:25.953+0000] {logging_mixin.py:188} INFO - Deleted unneeded file: /tmp/voiceover/.git/hooks/commit-msg.sample
[2025-04-06T15:26:25.955+0000] {logging_mixin.py:188} INFO - Deleted unneeded file: /tmp/voiceover/.git/hooks/pre-rebase.sample
[2025-04-06T15:26:25.957+0000] {logging_mixin.py:188} INFO - Deleted unneeded file: /tmp/voiceover/.git/hooks/update.sample
[2025-04-06T15:26:25.958+0000] {logging_mixin.py:188} INFO - Deleted unneeded file: /tmp/voiceover/.git/hooks/post-update.sample
[2025-04-06T15:26:25.959+0000] {logging_mixin.py:188} INFO - Deleted unneeded file: /tmp/voiceover/.git/hooks/pre-merge-commit.sample
[2025-04-06T15:26:25.959+0000] {logging_mixin.py:188} INFO - Deleted unneeded file: /tmp/voiceover/.git/hooks/prepare-commit-msg.sample
[2025-04-06T15:26:25.960+0000] {logging_mixin.py:188} INFO - Deleted unneeded file: /tmp/voiceover/.git/hooks/pre-applypatch.sample
[2025-04-06T15:26:25.960+0000] {logging_mixin.py:188} INFO - Deleted unneeded file: /tmp/voiceover/.git/hooks/applypatch-msg.sample
[2025-04-06T15:26:25.962+0000] {logging_mixin.py:188} INFO - Deleted unneeded file: /tmp/voiceover/.git/hooks/pre-push.sample
[2025-04-06T15:26:25.963+0000] {logging_mixin.py:188} INFO - Deleted unneeded file: /tmp/voiceover/.git/hooks/pre-commit.sample
[2025-04-06T15:26:25.964+0000] {logging_mixin.py:188} INFO - Deleted unneeded file: /tmp/voiceover/.git/info/exclude
[2025-04-06T15:26:25.965+0000] {logging_mixin.py:188} INFO - Deleted unneeded file: /tmp/voiceover/data/silence.mp3
[2025-04-06T15:26:25.966+0000] {logging_mixin.py:188} INFO - Deleted unneeded file: /tmp/voiceover/data/categorized.txt
[2025-04-06T15:26:25.966+0000] {logging_mixin.py:188} INFO - Deleted unneeded file: /tmp/voiceover/data/uncatagorize.txt
[2025-04-06T15:26:25.967+0000] {logging_mixin.py:188} WARNING - Processing voiceover: 23it [00:00, 268.75it/s]
[2025-04-06T15:26:25.970+0000] {logging_mixin.py:188} INFO - Deleted temporary folder: /tmp/voiceover
[2025-04-06T15:26:26.568+0000] {logging_mixin.py:188} WARNING - Processing crypto-pay: 0it [00:00, ?it/s]
[2025-04-06T15:26:26.572+0000] {logging_mixin.py:188} INFO - Deleted unneeded file: /tmp/crypto-pay/.git/HEAD
[2025-04-06T15:26:26.574+0000] {logging_mixin.py:188} INFO - Deleted unneeded file: /tmp/crypto-pay/.git/index
[2025-04-06T15:26:26.576+0000] {logging_mixin.py:188} INFO - Deleted unneeded file: /tmp/crypto-pay/.git/packed-refs
[2025-04-06T15:26:26.579+0000] {logging_mixin.py:188} INFO - Deleted unneeded file: /tmp/crypto-pay/.git/description
[2025-04-06T15:26:26.582+0000] {logging_mixin.py:188} INFO - Deleted unneeded file: /tmp/crypto-pay/.git/config
[2025-04-06T15:26:26.583+0000] {logging_mixin.py:188} INFO - Deleted unneeded file: /tmp/crypto-pay/.git/refs/remotes/origin/HEAD
[2025-04-06T15:26:26.585+0000] {logging_mixin.py:188} INFO - Deleted unneeded file: /tmp/crypto-pay/.git/refs/heads/main
[2025-04-06T15:26:26.588+0000] {logging_mixin.py:188} INFO - Deleted unneeded file: /tmp/crypto-pay/.git/logs/HEAD
[2025-04-06T15:26:26.590+0000] {logging_mixin.py:188} INFO - Deleted unneeded file: /tmp/crypto-pay/.git/logs/refs/remotes/origin/HEAD
[2025-04-06T15:26:26.591+0000] {logging_mixin.py:188} INFO - Deleted unneeded file: /tmp/crypto-pay/.git/logs/refs/heads/main
[2025-04-06T15:26:26.594+0000] {logging_mixin.py:188} INFO - Deleted unneeded file: /tmp/crypto-pay/.git/objects/pack/pack-65cbfc2d2224bfcf9ff0ed0443803ceb0d4ce9d2.pack
[2025-04-06T15:26:26.596+0000] {logging_mixin.py:188} INFO - Deleted unneeded file: /tmp/crypto-pay/.git/objects/pack/pack-65cbfc2d2224bfcf9ff0ed0443803ceb0d4ce9d2.idx
[2025-04-06T15:26:26.599+0000] {logging_mixin.py:188} INFO - Deleted unneeded file: /tmp/crypto-pay/.git/hooks/pre-receive.sample
[2025-04-06T15:26:26.605+0000] {logging_mixin.py:188} INFO - Deleted unneeded file: /tmp/crypto-pay/.git/hooks/fsmonitor-watchman.sample
[2025-04-06T15:26:26.608+0000] {logging_mixin.py:188} INFO - Deleted unneeded file: /tmp/crypto-pay/.git/hooks/push-to-checkout.sample
[2025-04-06T15:26:26.614+0000] {logging_mixin.py:188} INFO - Deleted unneeded file: /tmp/crypto-pay/.git/hooks/commit-msg.sample
[2025-04-06T15:26:26.620+0000] {logging_mixin.py:188} INFO - Deleted unneeded file: /tmp/crypto-pay/.git/hooks/pre-rebase.sample
[2025-04-06T15:26:26.624+0000] {logging_mixin.py:188} INFO - Deleted unneeded file: /tmp/crypto-pay/.git/hooks/update.sample
[2025-04-06T15:26:26.626+0000] {logging_mixin.py:188} INFO - Deleted unneeded file: /tmp/crypto-pay/.git/hooks/post-update.sample
[2025-04-06T15:26:26.630+0000] {logging_mixin.py:188} INFO - Deleted unneeded file: /tmp/crypto-pay/.git/hooks/pre-merge-commit.sample
[2025-04-06T15:26:26.632+0000] {logging_mixin.py:188} INFO - Deleted unneeded file: /tmp/crypto-pay/.git/hooks/prepare-commit-msg.sample
[2025-04-06T15:26:26.636+0000] {logging_mixin.py:188} INFO - Deleted unneeded file: /tmp/crypto-pay/.git/hooks/pre-applypatch.sample
[2025-04-06T15:26:26.640+0000] {logging_mixin.py:188} INFO - Deleted unneeded file: /tmp/crypto-pay/.git/hooks/applypatch-msg.sample
[2025-04-06T15:26:26.642+0000] {logging_mixin.py:188} INFO - Deleted unneeded file: /tmp/crypto-pay/.git/hooks/pre-push.sample
[2025-04-06T15:26:26.647+0000] {logging_mixin.py:188} INFO - Deleted unneeded file: /tmp/crypto-pay/.git/hooks/pre-commit.sample
[2025-04-06T15:26:26.649+0000] {logging_mixin.py:188} INFO - Deleted unneeded file: /tmp/crypto-pay/.git/info/exclude
[2025-04-06T15:26:26.655+0000] {logging_mixin.py:188} INFO - Deleted unneeded file: /tmp/crypto-pay/server/README.md
[2025-04-06T15:26:26.658+0000] {logging_mixin.py:188} INFO - Deleted unneeded file: /tmp/crypto-pay/server/docker-compose.yml
[2025-04-06T15:26:26.664+0000] {logging_mixin.py:188} INFO - Deleted unneeded file: /tmp/crypto-pay/server/.gitignore
[2025-04-06T15:26:26.665+0000] {logging_mixin.py:188} INFO - Deleted unneeded file: /tmp/crypto-pay/server/services/api_key/.dockerignore
[2025-04-06T15:26:26.673+0000] {_metadata.py:139} WARNING - Compute Engine Metadata server unavailable on attempt 1 of 3. Reason: [Errno 111] Connection refused
[2025-04-06T15:26:26.690+0000] {_metadata.py:139} WARNING - Compute Engine Metadata server unavailable on attempt 2 of 3. Reason: [Errno 111] Connection refused
[2025-04-06T15:26:26.712+0000] {_metadata.py:139} WARNING - Compute Engine Metadata server unavailable on attempt 3 of 3. Reason: [Errno 111] Connection refused
[2025-04-06T15:26:26.720+0000] {_default.py:338} WARNING - Authentication failed using Compute Engine authentication due to unavailable metadata server.
[2025-04-06T15:26:26.724+0000] {logging_mixin.py:188} INFO - Error initializing GCS client: Your default credentials were not found. To set up Application Default Credentials, see https://cloud.google.com/docs/authentication/external/set-up-adc for more information.
[2025-04-06T15:26:26.744+0000] {_metadata.py:139} WARNING - Compute Engine Metadata server unavailable on attempt 1 of 3. Reason: [Errno 111] Connection refused
[2025-04-06T15:26:26.757+0000] {_metadata.py:139} WARNING - Compute Engine Metadata server unavailable on attempt 2 of 3. Reason: [Errno 111] Connection refused
[2025-04-06T15:26:26.767+0000] {_metadata.py:139} WARNING - Compute Engine Metadata server unavailable on attempt 3 of 3. Reason: [Errno 111] Connection refused
[2025-04-06T15:26:26.770+0000] {_default.py:338} WARNING - Authentication failed using Compute Engine authentication due to unavailable metadata server.
[2025-04-06T15:26:26.775+0000] {logging_mixin.py:188} INFO - Error initializing GCS client: Your default credentials were not found. To set up Application Default Credentials, see https://cloud.google.com/docs/authentication/external/set-up-adc for more information.
[2025-04-06T15:26:26.778+0000] {logging_mixin.py:188} INFO - Deleted unneeded file: /tmp/crypto-pay/server/services/api_key/Dockerfile
[2025-04-06T15:26:26.798+0000] {_metadata.py:139} WARNING - Compute Engine Metadata server unavailable on attempt 1 of 3. Reason: [Errno 111] Connection refused
[2025-04-06T15:26:26.811+0000] {_metadata.py:139} WARNING - Compute Engine Metadata server unavailable on attempt 2 of 3. Reason: [Errno 111] Connection refused
[2025-04-06T15:26:26.822+0000] {_metadata.py:139} WARNING - Compute Engine Metadata server unavailable on attempt 3 of 3. Reason: [Errno 111] Connection refused
[2025-04-06T15:26:26.828+0000] {_default.py:338} WARNING - Authentication failed using Compute Engine authentication due to unavailable metadata server.
[2025-04-06T15:26:26.835+0000] {logging_mixin.py:188} INFO - Error initializing GCS client: Your default credentials were not found. To set up Application Default Credentials, see https://cloud.google.com/docs/authentication/external/set-up-adc for more information.
[2025-04-06T15:26:26.845+0000] {logging_mixin.py:188} WARNING - Processing crypto-pay: 21it [00:00, 77.06it/s]
[2025-04-06T15:26:26.862+0000] {_metadata.py:139} WARNING - Compute Engine Metadata server unavailable on attempt 1 of 3. Reason: [Errno 111] Connection refused
[2025-04-06T15:26:26.877+0000] {_metadata.py:139} WARNING - Compute Engine Metadata server unavailable on attempt 2 of 3. Reason: [Errno 111] Connection refused
[2025-04-06T15:26:26.926+0000] {_metadata.py:139} WARNING - Compute Engine Metadata server unavailable on attempt 3 of 3. Reason: [Errno 111] Connection refused
[2025-04-06T15:26:26.974+0000] {_default.py:338} WARNING - Authentication failed using Compute Engine authentication due to unavailable metadata server.
[2025-04-06T15:26:26.982+0000] {logging_mixin.py:188} INFO - Error initializing GCS client: Your default credentials were not found. To set up Application Default Credentials, see https://cloud.google.com/docs/authentication/external/set-up-adc for more information.
[2025-04-06T15:26:27.016+0000] {_metadata.py:139} WARNING - Compute Engine Metadata server unavailable on attempt 1 of 3. Reason: [Errno 111] Connection refused
[2025-04-06T15:26:27.043+0000] {_metadata.py:139} WARNING - Compute Engine Metadata server unavailable on attempt 2 of 3. Reason: [Errno 111] Connection refused
[2025-04-06T15:26:27.053+0000] {_metadata.py:139} WARNING - Compute Engine Metadata server unavailable on attempt 3 of 3. Reason: [Errno 111] Connection refused
[2025-04-06T15:26:27.055+0000] {_default.py:338} WARNING - Authentication failed using Compute Engine authentication due to unavailable metadata server.
[2025-04-06T15:26:27.062+0000] {logging_mixin.py:188} INFO - Error initializing GCS client: Your default credentials were not found. To set up Application Default Credentials, see https://cloud.google.com/docs/authentication/external/set-up-adc for more information.
[2025-04-06T15:26:27.072+0000] {_metadata.py:139} WARNING - Compute Engine Metadata server unavailable on attempt 1 of 3. Reason: [Errno 111] Connection refused
[2025-04-06T15:26:27.085+0000] {_metadata.py:139} WARNING - Compute Engine Metadata server unavailable on attempt 2 of 3. Reason: [Errno 111] Connection refused
[2025-04-06T15:26:27.094+0000] {_metadata.py:139} WARNING - Compute Engine Metadata server unavailable on attempt 3 of 3. Reason: [Errno 111] Connection refused
[2025-04-06T15:26:27.096+0000] {_default.py:338} WARNING - Authentication failed using Compute Engine authentication due to unavailable metadata server.
[2025-04-06T15:26:27.100+0000] {logging_mixin.py:188} INFO - Error initializing GCS client: Your default credentials were not found. To set up Application Default Credentials, see https://cloud.google.com/docs/authentication/external/set-up-adc for more information.
[2025-04-06T15:26:27.103+0000] {logging_mixin.py:188} INFO - Deleted unneeded file: /tmp/crypto-pay/server/services/auth/README.md
[2025-04-06T15:26:27.112+0000] {logging_mixin.py:188} INFO - Deleted unneeded file: /tmp/crypto-pay/server/services/auth/package.json
[2025-04-06T15:26:27.113+0000] {logging_mixin.py:188} INFO - Deleted unneeded file: /tmp/crypto-pay/server/services/auth/.dockerignore
[2025-04-06T15:26:27.114+0000] {logging_mixin.py:188} INFO - Deleted unneeded file: /tmp/crypto-pay/server/services/auth/app.js
[2025-04-06T15:26:27.116+0000] {logging_mixin.py:188} INFO - Deleted unneeded file: /tmp/crypto-pay/server/services/auth/Dockerfile
[2025-04-06T15:26:27.121+0000] {logging_mixin.py:188} INFO - Deleted unneeded file: /tmp/crypto-pay/server/services/auth/api/middlewares.js
[2025-04-06T15:26:27.122+0000] {logging_mixin.py:188} INFO - Deleted unneeded file: /tmp/crypto-pay/server/services/auth/api/authController.js
[2025-04-06T15:26:27.124+0000] {logging_mixin.py:188} INFO - Deleted unneeded file: /tmp/crypto-pay/server/services/auth/api/authModel.js
[2025-04-06T15:26:27.125+0000] {logging_mixin.py:188} INFO - Deleted unneeded file: /tmp/crypto-pay/server/services/auth/api/authRoutes.js
[2025-04-06T15:26:27.126+0000] {logging_mixin.py:188} WARNING - Processing crypto-pay: 24it [00:00, 43.29it/s]
[2025-04-06T15:26:27.131+0000] {logging_mixin.py:188} INFO - Deleted temporary folder: /tmp/crypto-pay
[2025-04-06T15:26:27.858+0000] {logging_mixin.py:188} WARNING - Processing SpeakEasy-ChatBot: 0it [00:00, ?it/s]
[2025-04-06T15:26:27.874+0000] {_metadata.py:139} WARNING - Compute Engine Metadata server unavailable on attempt 1 of 3. Reason: [Errno 111] Connection refused
[2025-04-06T15:26:27.882+0000] {_metadata.py:139} WARNING - Compute Engine Metadata server unavailable on attempt 2 of 3. Reason: [Errno 111] Connection refused
[2025-04-06T15:26:27.885+0000] {_metadata.py:139} WARNING - Compute Engine Metadata server unavailable on attempt 3 of 3. Reason: [Errno 111] Connection refused
[2025-04-06T15:26:27.886+0000] {_default.py:338} WARNING - Authentication failed using Compute Engine authentication due to unavailable metadata server.
[2025-04-06T15:26:27.886+0000] {logging_mixin.py:188} INFO - Error initializing GCS client: Your default credentials were not found. To set up Application Default Credentials, see https://cloud.google.com/docs/authentication/external/set-up-adc for more information.
[2025-04-06T15:26:27.887+0000] {logging_mixin.py:188} INFO - Deleted unneeded file: /tmp/SpeakEasy-ChatBot/README.md
[2025-04-06T15:26:27.888+0000] {logging_mixin.py:188} INFO - Deleted unneeded file: /tmp/SpeakEasy-ChatBot/requirements.txt
[2025-04-06T15:26:27.893+0000] {_metadata.py:139} WARNING - Compute Engine Metadata server unavailable on attempt 1 of 3. Reason: [Errno 111] Connection refused
[2025-04-06T15:26:27.897+0000] {_metadata.py:139} WARNING - Compute Engine Metadata server unavailable on attempt 2 of 3. Reason: [Errno 111] Connection refused
[2025-04-06T15:26:27.901+0000] {_metadata.py:139} WARNING - Compute Engine Metadata server unavailable on attempt 3 of 3. Reason: [Errno 111] Connection refused
[2025-04-06T15:26:27.903+0000] {_default.py:338} WARNING - Authentication failed using Compute Engine authentication due to unavailable metadata server.
[2025-04-06T15:26:27.905+0000] {logging_mixin.py:188} INFO - Error initializing GCS client: Your default credentials were not found. To set up Application Default Credentials, see https://cloud.google.com/docs/authentication/external/set-up-adc for more information.
[2025-04-06T15:26:27.906+0000] {logging_mixin.py:188} INFO - Deleted unneeded file: /tmp/SpeakEasy-ChatBot/.git/HEAD
[2025-04-06T15:26:27.907+0000] {logging_mixin.py:188} INFO - Deleted unneeded file: /tmp/SpeakEasy-ChatBot/.git/index
[2025-04-06T15:26:27.909+0000] {logging_mixin.py:188} INFO - Deleted unneeded file: /tmp/SpeakEasy-ChatBot/.git/packed-refs
[2025-04-06T15:26:27.909+0000] {logging_mixin.py:188} INFO - Deleted unneeded file: /tmp/SpeakEasy-ChatBot/.git/description
[2025-04-06T15:26:27.910+0000] {logging_mixin.py:188} INFO - Deleted unneeded file: /tmp/SpeakEasy-ChatBot/.git/config
[2025-04-06T15:26:27.912+0000] {logging_mixin.py:188} INFO - Deleted unneeded file: /tmp/SpeakEasy-ChatBot/.git/refs/remotes/origin/HEAD
[2025-04-06T15:26:27.913+0000] {logging_mixin.py:188} INFO - Deleted unneeded file: /tmp/SpeakEasy-ChatBot/.git/refs/heads/AIDS
[2025-04-06T15:26:27.915+0000] {logging_mixin.py:188} INFO - Deleted unneeded file: /tmp/SpeakEasy-ChatBot/.git/logs/HEAD
[2025-04-06T15:26:27.916+0000] {logging_mixin.py:188} INFO - Deleted unneeded file: /tmp/SpeakEasy-ChatBot/.git/logs/refs/remotes/origin/HEAD
[2025-04-06T15:26:27.917+0000] {logging_mixin.py:188} INFO - Deleted unneeded file: /tmp/SpeakEasy-ChatBot/.git/logs/refs/heads/AIDS
[2025-04-06T15:26:27.917+0000] {logging_mixin.py:188} INFO - Deleted unneeded file: /tmp/SpeakEasy-ChatBot/.git/objects/pack/pack-a4db57afe81535fb5000c54b4a9c2d624c327ea2.pack
[2025-04-06T15:26:27.918+0000] {logging_mixin.py:188} INFO - Deleted unneeded file: /tmp/SpeakEasy-ChatBot/.git/objects/pack/pack-a4db57afe81535fb5000c54b4a9c2d624c327ea2.idx
[2025-04-06T15:26:27.919+0000] {logging_mixin.py:188} INFO - Deleted unneeded file: /tmp/SpeakEasy-ChatBot/.git/hooks/pre-receive.sample
[2025-04-06T15:26:27.921+0000] {logging_mixin.py:188} INFO - Deleted unneeded file: /tmp/SpeakEasy-ChatBot/.git/hooks/fsmonitor-watchman.sample
[2025-04-06T15:26:27.922+0000] {logging_mixin.py:188} INFO - Deleted unneeded file: /tmp/SpeakEasy-ChatBot/.git/hooks/push-to-checkout.sample
[2025-04-06T15:26:27.923+0000] {logging_mixin.py:188} INFO - Deleted unneeded file: /tmp/SpeakEasy-ChatBot/.git/hooks/commit-msg.sample
[2025-04-06T15:26:27.924+0000] {logging_mixin.py:188} INFO - Deleted unneeded file: /tmp/SpeakEasy-ChatBot/.git/hooks/pre-rebase.sample
[2025-04-06T15:26:27.924+0000] {logging_mixin.py:188} INFO - Deleted unneeded file: /tmp/SpeakEasy-ChatBot/.git/hooks/update.sample
[2025-04-06T15:26:27.925+0000] {logging_mixin.py:188} INFO - Deleted unneeded file: /tmp/SpeakEasy-ChatBot/.git/hooks/post-update.sample
[2025-04-06T15:26:27.926+0000] {logging_mixin.py:188} INFO - Deleted unneeded file: /tmp/SpeakEasy-ChatBot/.git/hooks/pre-merge-commit.sample
[2025-04-06T15:26:27.927+0000] {logging_mixin.py:188} INFO - Deleted unneeded file: /tmp/SpeakEasy-ChatBot/.git/hooks/prepare-commit-msg.sample
[2025-04-06T15:26:27.928+0000] {logging_mixin.py:188} INFO - Deleted unneeded file: /tmp/SpeakEasy-ChatBot/.git/hooks/pre-applypatch.sample
[2025-04-06T15:26:27.928+0000] {logging_mixin.py:188} INFO - Deleted unneeded file: /tmp/SpeakEasy-ChatBot/.git/hooks/applypatch-msg.sample
[2025-04-06T15:26:27.929+0000] {logging_mixin.py:188} INFO - Deleted unneeded file: /tmp/SpeakEasy-ChatBot/.git/hooks/pre-push.sample
[2025-04-06T15:26:27.930+0000] {logging_mixin.py:188} INFO - Deleted unneeded file: /tmp/SpeakEasy-ChatBot/.git/hooks/pre-commit.sample
[2025-04-06T15:26:27.931+0000] {logging_mixin.py:188} INFO - Deleted unneeded file: /tmp/SpeakEasy-ChatBot/.git/info/exclude
[2025-04-06T15:26:27.934+0000] {_metadata.py:139} WARNING - Compute Engine Metadata server unavailable on attempt 1 of 3. Reason: [Errno 111] Connection refused
[2025-04-06T15:26:27.938+0000] {_metadata.py:139} WARNING - Compute Engine Metadata server unavailable on attempt 2 of 3. Reason: [Errno 111] Connection refused
[2025-04-06T15:26:27.942+0000] {_metadata.py:139} WARNING - Compute Engine Metadata server unavailable on attempt 3 of 3. Reason: [Errno 111] Connection refused
[2025-04-06T15:26:27.943+0000] {_default.py:338} WARNING - Authentication failed using Compute Engine authentication due to unavailable metadata server.
[2025-04-06T15:26:27.944+0000] {logging_mixin.py:188} INFO - Error initializing GCS client: Your default credentials were not found. To set up Application Default Credentials, see https://cloud.google.com/docs/authentication/external/set-up-adc for more information.
[2025-04-06T15:26:27.947+0000] {_metadata.py:139} WARNING - Compute Engine Metadata server unavailable on attempt 1 of 3. Reason: [Errno 111] Connection refused
[2025-04-06T15:26:27.951+0000] {_metadata.py:139} WARNING - Compute Engine Metadata server unavailable on attempt 2 of 3. Reason: [Errno 111] Connection refused
[2025-04-06T15:26:27.954+0000] {_metadata.py:139} WARNING - Compute Engine Metadata server unavailable on attempt 3 of 3. Reason: [Errno 111] Connection refused
[2025-04-06T15:26:27.955+0000] {_default.py:338} WARNING - Authentication failed using Compute Engine authentication due to unavailable metadata server.
[2025-04-06T15:26:27.956+0000] {logging_mixin.py:188} INFO - Error initializing GCS client: Your default credentials were not found. To set up Application Default Credentials, see https://cloud.google.com/docs/authentication/external/set-up-adc for more information.
[2025-04-06T15:26:27.957+0000] {logging_mixin.py:188} INFO - Deleted unneeded file: /tmp/SpeakEasy-ChatBot/.github/workflows/python-app.yml
[2025-04-06T15:26:27.958+0000] {logging_mixin.py:188} WARNING - Processing SpeakEasy-ChatBot: 22it [00:00, 225.66it/s]
[2025-04-06T15:26:27.961+0000] {logging_mixin.py:188} INFO - Deleted temporary folder: /tmp/SpeakEasy-ChatBot
[2025-04-06T15:26:28.573+0000] {logging_mixin.py:188} WARNING - Processing codepad: 0it [00:00, ?it/s]
[2025-04-06T15:26:28.577+0000] {_metadata.py:139} WARNING - Compute Engine Metadata server unavailable on attempt 1 of 3. Reason: [Errno 111] Connection refused
[2025-04-06T15:26:28.581+0000] {_metadata.py:139} WARNING - Compute Engine Metadata server unavailable on attempt 2 of 3. Reason: [Errno 111] Connection refused
[2025-04-06T15:26:28.584+0000] {_metadata.py:139} WARNING - Compute Engine Metadata server unavailable on attempt 3 of 3. Reason: [Errno 111] Connection refused
[2025-04-06T15:26:28.585+0000] {_default.py:338} WARNING - Authentication failed using Compute Engine authentication due to unavailable metadata server.
[2025-04-06T15:26:28.586+0000] {logging_mixin.py:188} INFO - Error initializing GCS client: Your default credentials were not found. To set up Application Default Credentials, see https://cloud.google.com/docs/authentication/external/set-up-adc for more information.
[2025-04-06T15:26:28.587+0000] {logging_mixin.py:188} INFO - Deleted unneeded file: /tmp/codepad/README.md
[2025-04-06T15:26:28.587+0000] {logging_mixin.py:188} INFO - Deleted unneeded file: /tmp/codepad/package.json
[2025-04-06T15:26:28.588+0000] {logging_mixin.py:188} INFO - Deleted unneeded file: /tmp/codepad/.gitignore
[2025-04-06T15:26:28.588+0000] {logging_mixin.py:188} INFO - Deleted unneeded file: /tmp/codepad/server.js
[2025-04-06T15:26:28.592+0000] {_metadata.py:139} WARNING - Compute Engine Metadata server unavailable on attempt 1 of 3. Reason: [Errno 111] Connection refused
[2025-04-06T15:26:28.595+0000] {_metadata.py:139} WARNING - Compute Engine Metadata server unavailable on attempt 2 of 3. Reason: [Errno 111] Connection refused
[2025-04-06T15:26:28.598+0000] {_metadata.py:139} WARNING - Compute Engine Metadata server unavailable on attempt 3 of 3. Reason: [Errno 111] Connection refused
[2025-04-06T15:26:28.600+0000] {_default.py:338} WARNING - Authentication failed using Compute Engine authentication due to unavailable metadata server.
[2025-04-06T15:26:28.601+0000] {logging_mixin.py:188} INFO - Error initializing GCS client: Your default credentials were not found. To set up Application Default Credentials, see https://cloud.google.com/docs/authentication/external/set-up-adc for more information.
[2025-04-06T15:26:28.602+0000] {logging_mixin.py:188} INFO - Deleted unneeded file: /tmp/codepad/package-lock.json
[2025-04-06T15:26:28.603+0000] {logging_mixin.py:188} INFO - Deleted unneeded file: /tmp/codepad/main.js
[2025-04-06T15:26:28.608+0000] {_metadata.py:139} WARNING - Compute Engine Metadata server unavailable on attempt 1 of 3. Reason: [Errno 111] Connection refused
[2025-04-06T15:26:28.612+0000] {_metadata.py:139} WARNING - Compute Engine Metadata server unavailable on attempt 2 of 3. Reason: [Errno 111] Connection refused
[2025-04-06T15:26:28.615+0000] {_metadata.py:139} WARNING - Compute Engine Metadata server unavailable on attempt 3 of 3. Reason: [Errno 111] Connection refused
[2025-04-06T15:26:28.616+0000] {_default.py:338} WARNING - Authentication failed using Compute Engine authentication due to unavailable metadata server.
[2025-04-06T15:26:28.616+0000] {logging_mixin.py:188} INFO - Error initializing GCS client: Your default credentials were not found. To set up Application Default Credentials, see https://cloud.google.com/docs/authentication/external/set-up-adc for more information.
[2025-04-06T15:26:28.617+0000] {logging_mixin.py:188} INFO - Deleted unneeded file: /tmp/codepad/.git/HEAD
[2025-04-06T15:26:28.618+0000] {logging_mixin.py:188} INFO - Deleted unneeded file: /tmp/codepad/.git/index
[2025-04-06T15:26:28.619+0000] {logging_mixin.py:188} INFO - Deleted unneeded file: /tmp/codepad/.git/packed-refs
[2025-04-06T15:26:28.620+0000] {logging_mixin.py:188} INFO - Deleted unneeded file: /tmp/codepad/.git/description
[2025-04-06T15:26:28.620+0000] {logging_mixin.py:188} INFO - Deleted unneeded file: /tmp/codepad/.git/config
[2025-04-06T15:26:28.622+0000] {logging_mixin.py:188} INFO - Deleted unneeded file: /tmp/codepad/.git/refs/remotes/origin/HEAD
[2025-04-06T15:26:28.623+0000] {logging_mixin.py:188} INFO - Deleted unneeded file: /tmp/codepad/.git/refs/heads/main
[2025-04-06T15:26:28.626+0000] {logging_mixin.py:188} INFO - Deleted unneeded file: /tmp/codepad/.git/logs/HEAD
[2025-04-06T15:26:28.627+0000] {logging_mixin.py:188} INFO - Deleted unneeded file: /tmp/codepad/.git/logs/refs/remotes/origin/HEAD
[2025-04-06T15:26:28.628+0000] {logging_mixin.py:188} INFO - Deleted unneeded file: /tmp/codepad/.git/logs/refs/heads/main
[2025-04-06T15:26:28.629+0000] {logging_mixin.py:188} INFO - Deleted unneeded file: /tmp/codepad/.git/objects/pack/pack-8fdb65d4293c6720cc189f1baf38c61811eda3c8.pack
[2025-04-06T15:26:28.630+0000] {logging_mixin.py:188} INFO - Deleted unneeded file: /tmp/codepad/.git/objects/pack/pack-8fdb65d4293c6720cc189f1baf38c61811eda3c8.idx
[2025-04-06T15:26:28.632+0000] {logging_mixin.py:188} INFO - Deleted unneeded file: /tmp/codepad/.git/hooks/pre-receive.sample
[2025-04-06T15:26:28.633+0000] {logging_mixin.py:188} INFO - Deleted unneeded file: /tmp/codepad/.git/hooks/fsmonitor-watchman.sample
[2025-04-06T15:26:28.634+0000] {logging_mixin.py:188} INFO - Deleted unneeded file: /tmp/codepad/.git/hooks/push-to-checkout.sample
[2025-04-06T15:26:28.635+0000] {logging_mixin.py:188} INFO - Deleted unneeded file: /tmp/codepad/.git/hooks/commit-msg.sample
[2025-04-06T15:26:28.636+0000] {logging_mixin.py:188} INFO - Deleted unneeded file: /tmp/codepad/.git/hooks/pre-rebase.sample
[2025-04-06T15:26:28.637+0000] {logging_mixin.py:188} INFO - Deleted unneeded file: /tmp/codepad/.git/hooks/update.sample
[2025-04-06T15:26:28.638+0000] {logging_mixin.py:188} INFO - Deleted unneeded file: /tmp/codepad/.git/hooks/post-update.sample
[2025-04-06T15:26:28.639+0000] {logging_mixin.py:188} INFO - Deleted unneeded file: /tmp/codepad/.git/hooks/pre-merge-commit.sample
[2025-04-06T15:26:28.640+0000] {logging_mixin.py:188} INFO - Deleted unneeded file: /tmp/codepad/.git/hooks/prepare-commit-msg.sample
[2025-04-06T15:26:28.641+0000] {logging_mixin.py:188} INFO - Deleted unneeded file: /tmp/codepad/.git/hooks/pre-applypatch.sample
[2025-04-06T15:26:28.642+0000] {logging_mixin.py:188} INFO - Deleted unneeded file: /tmp/codepad/.git/hooks/applypatch-msg.sample
[2025-04-06T15:26:28.643+0000] {logging_mixin.py:188} INFO - Deleted unneeded file: /tmp/codepad/.git/hooks/pre-push.sample
[2025-04-06T15:26:28.644+0000] {logging_mixin.py:188} INFO - Deleted unneeded file: /tmp/codepad/.git/hooks/pre-commit.sample
[2025-04-06T15:26:28.645+0000] {logging_mixin.py:188} INFO - Deleted unneeded file: /tmp/codepad/.git/info/exclude
[2025-04-06T15:26:28.646+0000] {logging_mixin.py:188} WARNING - Processing codepad: 19it [00:00, 264.40it/s]
[2025-04-06T15:26:28.648+0000] {logging_mixin.py:188} INFO - Deleted temporary folder: /tmp/codepad
[2025-04-06T15:26:29.424+0000] {logging_mixin.py:188} WARNING - Processing aarinmahala: 0it [00:00, ?it/s]
[2025-04-06T15:26:29.427+0000] {_metadata.py:139} WARNING - Compute Engine Metadata server unavailable on attempt 1 of 3. Reason: [Errno 111] Connection refused
[2025-04-06T15:26:29.430+0000] {_metadata.py:139} WARNING - Compute Engine Metadata server unavailable on attempt 2 of 3. Reason: [Errno 111] Connection refused
[2025-04-06T15:26:29.433+0000] {_metadata.py:139} WARNING - Compute Engine Metadata server unavailable on attempt 3 of 3. Reason: [Errno 111] Connection refused
[2025-04-06T15:26:29.434+0000] {_default.py:338} WARNING - Authentication failed using Compute Engine authentication due to unavailable metadata server.
[2025-04-06T15:26:29.434+0000] {logging_mixin.py:188} INFO - Error initializing GCS client: Your default credentials were not found. To set up Application Default Credentials, see https://cloud.google.com/docs/authentication/external/set-up-adc for more information.
[2025-04-06T15:26:29.437+0000] {_metadata.py:139} WARNING - Compute Engine Metadata server unavailable on attempt 1 of 3. Reason: [Errno 111] Connection refused
[2025-04-06T15:26:29.440+0000] {_metadata.py:139} WARNING - Compute Engine Metadata server unavailable on attempt 2 of 3. Reason: [Errno 111] Connection refused
[2025-04-06T15:26:29.442+0000] {_metadata.py:139} WARNING - Compute Engine Metadata server unavailable on attempt 3 of 3. Reason: [Errno 111] Connection refused
[2025-04-06T15:26:29.443+0000] {_default.py:338} WARNING - Authentication failed using Compute Engine authentication due to unavailable metadata server.
[2025-04-06T15:26:29.443+0000] {logging_mixin.py:188} INFO - Error initializing GCS client: Your default credentials were not found. To set up Application Default Credentials, see https://cloud.google.com/docs/authentication/external/set-up-adc for more information.
[2025-04-06T15:26:29.444+0000] {logging_mixin.py:188} INFO - Deleted unneeded file: /tmp/aarinmahala/Unconfirmed 929645.crdownload
[2025-04-06T15:26:29.445+0000] {logging_mixin.py:188} INFO - Deleted unneeded file: /tmp/aarinmahala/im11.jpg
[2025-04-06T15:26:29.446+0000] {logging_mixin.py:188} INFO - Deleted unneeded file: /tmp/aarinmahala/script.js
[2025-04-06T15:26:29.447+0000] {logging_mixin.py:188} INFO - Deleted unneeded file: /tmp/aarinmahala/im1.jpg
[2025-04-06T15:26:29.447+0000] {logging_mixin.py:188} INFO - Deleted unneeded file: /tmp/aarinmahala/.git/HEAD
[2025-04-06T15:26:29.448+0000] {logging_mixin.py:188} INFO - Deleted unneeded file: /tmp/aarinmahala/.git/index
[2025-04-06T15:26:29.448+0000] {logging_mixin.py:188} INFO - Deleted unneeded file: /tmp/aarinmahala/.git/packed-refs
[2025-04-06T15:26:29.449+0000] {logging_mixin.py:188} INFO - Deleted unneeded file: /tmp/aarinmahala/.git/description
[2025-04-06T15:26:29.449+0000] {logging_mixin.py:188} INFO - Deleted unneeded file: /tmp/aarinmahala/.git/config
[2025-04-06T15:26:29.450+0000] {logging_mixin.py:188} INFO - Deleted unneeded file: /tmp/aarinmahala/.git/refs/remotes/origin/HEAD
[2025-04-06T15:26:29.451+0000] {logging_mixin.py:188} INFO - Deleted unneeded file: /tmp/aarinmahala/.git/refs/heads/main
[2025-04-06T15:26:29.451+0000] {logging_mixin.py:188} INFO - Deleted unneeded file: /tmp/aarinmahala/.git/logs/HEAD
[2025-04-06T15:26:29.452+0000] {logging_mixin.py:188} INFO - Deleted unneeded file: /tmp/aarinmahala/.git/logs/refs/remotes/origin/HEAD
[2025-04-06T15:26:29.452+0000] {logging_mixin.py:188} INFO - Deleted unneeded file: /tmp/aarinmahala/.git/logs/refs/heads/main
[2025-04-06T15:26:29.453+0000] {logging_mixin.py:188} INFO - Deleted unneeded file: /tmp/aarinmahala/.git/objects/pack/pack-f3892bb5777262aacedfa289986fc151aeea249a.pack
[2025-04-06T15:26:29.454+0000] {logging_mixin.py:188} INFO - Deleted unneeded file: /tmp/aarinmahala/.git/objects/pack/pack-f3892bb5777262aacedfa289986fc151aeea249a.idx
[2025-04-06T15:26:29.454+0000] {logging_mixin.py:188} INFO - Deleted unneeded file: /tmp/aarinmahala/.git/hooks/pre-receive.sample
[2025-04-06T15:26:29.455+0000] {logging_mixin.py:188} INFO - Deleted unneeded file: /tmp/aarinmahala/.git/hooks/fsmonitor-watchman.sample
[2025-04-06T15:26:29.455+0000] {logging_mixin.py:188} INFO - Deleted unneeded file: /tmp/aarinmahala/.git/hooks/push-to-checkout.sample
[2025-04-06T15:26:29.456+0000] {logging_mixin.py:188} INFO - Deleted unneeded file: /tmp/aarinmahala/.git/hooks/commit-msg.sample
[2025-04-06T15:26:29.456+0000] {logging_mixin.py:188} INFO - Deleted unneeded file: /tmp/aarinmahala/.git/hooks/pre-rebase.sample
[2025-04-06T15:26:29.457+0000] {logging_mixin.py:188} INFO - Deleted unneeded file: /tmp/aarinmahala/.git/hooks/update.sample
[2025-04-06T15:26:29.457+0000] {logging_mixin.py:188} INFO - Deleted unneeded file: /tmp/aarinmahala/.git/hooks/post-update.sample
[2025-04-06T15:26:29.458+0000] {logging_mixin.py:188} INFO - Deleted unneeded file: /tmp/aarinmahala/.git/hooks/pre-merge-commit.sample
[2025-04-06T15:26:29.458+0000] {logging_mixin.py:188} INFO - Deleted unneeded file: /tmp/aarinmahala/.git/hooks/prepare-commit-msg.sample
[2025-04-06T15:26:29.459+0000] {logging_mixin.py:188} INFO - Deleted unneeded file: /tmp/aarinmahala/.git/hooks/pre-applypatch.sample
[2025-04-06T15:26:29.459+0000] {logging_mixin.py:188} INFO - Deleted unneeded file: /tmp/aarinmahala/.git/hooks/applypatch-msg.sample
[2025-04-06T15:26:29.460+0000] {logging_mixin.py:188} INFO - Deleted unneeded file: /tmp/aarinmahala/.git/hooks/pre-push.sample
[2025-04-06T15:26:29.460+0000] {logging_mixin.py:188} INFO - Deleted unneeded file: /tmp/aarinmahala/.git/hooks/pre-commit.sample
[2025-04-06T15:26:29.461+0000] {logging_mixin.py:188} INFO - Deleted unneeded file: /tmp/aarinmahala/.git/info/exclude
[2025-04-06T15:26:29.463+0000] {logging_mixin.py:188} WARNING - Processing aarinmahala: 18it [00:00, 470.13it/s]
[2025-04-06T15:26:29.465+0000] {logging_mixin.py:188} INFO - Deleted temporary folder: /tmp/aarinmahala
[2025-04-06T15:26:29.466+0000] {logging_mixin.py:188} INFO - Repository webweather already cloned. Skipping.
[2025-04-06T15:26:30.069+0000] {logging_mixin.py:188} WARNING - Processing skills: 0it [00:00, ?it/s]
[2025-04-06T15:26:30.071+0000] {logging_mixin.py:188} INFO - Deleted unneeded file: /tmp/skills/README.md
[2025-04-06T15:26:30.073+0000] {logging_mixin.py:188} INFO - Deleted unneeded file: /tmp/skills/.gitignore
[2025-04-06T15:26:30.075+0000] {logging_mixin.py:188} INFO - Deleted unneeded file: /tmp/skills/.idea/material_theme_project_new.xml
[2025-04-06T15:26:30.076+0000] {logging_mixin.py:188} INFO - Deleted unneeded file: /tmp/skills/.idea/vcs.xml
[2025-04-06T15:26:30.076+0000] {logging_mixin.py:188} INFO - Deleted unneeded file: /tmp/skills/.idea/skills.iml
[2025-04-06T15:26:30.077+0000] {logging_mixin.py:188} INFO - Deleted unneeded file: /tmp/skills/.idea/.gitignore
[2025-04-06T15:26:30.079+0000] {logging_mixin.py:188} INFO - Deleted unneeded file: /tmp/skills/.idea/modules.xml
[2025-04-06T15:26:30.081+0000] {logging_mixin.py:188} INFO - Deleted unneeded file: /tmp/skills/JS/scripts/AverageCalculator.js
[2025-04-06T15:26:30.084+0000] {logging_mixin.py:188} INFO - Deleted unneeded file: /tmp/skills/JS/scripts/FrequencyCalculator.js
[2025-04-06T15:26:30.085+0000] {logging_mixin.py:188} INFO - Deleted unneeded file: /tmp/skills/.git/HEAD
[2025-04-06T15:26:30.088+0000] {logging_mixin.py:188} INFO - Deleted unneeded file: /tmp/skills/.git/index
[2025-04-06T15:26:30.089+0000] {logging_mixin.py:188} INFO - Deleted unneeded file: /tmp/skills/.git/packed-refs
[2025-04-06T15:26:30.091+0000] {logging_mixin.py:188} INFO - Deleted unneeded file: /tmp/skills/.git/description
[2025-04-06T15:26:30.092+0000] {logging_mixin.py:188} INFO - Deleted unneeded file: /tmp/skills/.git/config
[2025-04-06T15:26:30.093+0000] {logging_mixin.py:188} INFO - Deleted unneeded file: /tmp/skills/.git/refs/remotes/origin/HEAD
[2025-04-06T15:26:30.094+0000] {logging_mixin.py:188} INFO - Deleted unneeded file: /tmp/skills/.git/refs/heads/main
[2025-04-06T15:26:30.095+0000] {logging_mixin.py:188} INFO - Deleted unneeded file: /tmp/skills/.git/logs/HEAD
[2025-04-06T15:26:30.097+0000] {logging_mixin.py:188} INFO - Deleted unneeded file: /tmp/skills/.git/logs/refs/remotes/origin/HEAD
[2025-04-06T15:26:30.097+0000] {logging_mixin.py:188} INFO - Deleted unneeded file: /tmp/skills/.git/logs/refs/heads/main
[2025-04-06T15:26:30.099+0000] {logging_mixin.py:188} INFO - Deleted unneeded file: /tmp/skills/.git/objects/pack/pack-2246a1f5eacd60ad0ae599fba19b557c9bfeb6ac.pack
[2025-04-06T15:26:30.101+0000] {logging_mixin.py:188} INFO - Deleted unneeded file: /tmp/skills/.git/objects/pack/pack-2246a1f5eacd60ad0ae599fba19b557c9bfeb6ac.idx
[2025-04-06T15:26:30.102+0000] {logging_mixin.py:188} INFO - Deleted unneeded file: /tmp/skills/.git/hooks/pre-receive.sample
[2025-04-06T15:26:30.103+0000] {logging_mixin.py:188} INFO - Deleted unneeded file: /tmp/skills/.git/hooks/fsmonitor-watchman.sample
[2025-04-06T15:26:30.104+0000] {logging_mixin.py:188} INFO - Deleted unneeded file: /tmp/skills/.git/hooks/push-to-checkout.sample
[2025-04-06T15:26:30.105+0000] {logging_mixin.py:188} INFO - Deleted unneeded file: /tmp/skills/.git/hooks/commit-msg.sample
[2025-04-06T15:26:30.105+0000] {logging_mixin.py:188} INFO - Deleted unneeded file: /tmp/skills/.git/hooks/pre-rebase.sample
[2025-04-06T15:26:30.107+0000] {logging_mixin.py:188} INFO - Deleted unneeded file: /tmp/skills/.git/hooks/update.sample
[2025-04-06T15:26:30.108+0000] {logging_mixin.py:188} INFO - Deleted unneeded file: /tmp/skills/.git/hooks/post-update.sample
[2025-04-06T15:26:30.109+0000] {logging_mixin.py:188} INFO - Deleted unneeded file: /tmp/skills/.git/hooks/pre-merge-commit.sample
[2025-04-06T15:26:30.110+0000] {logging_mixin.py:188} INFO - Deleted unneeded file: /tmp/skills/.git/hooks/prepare-commit-msg.sample
[2025-04-06T15:26:30.110+0000] {logging_mixin.py:188} INFO - Deleted unneeded file: /tmp/skills/.git/hooks/pre-applypatch.sample
[2025-04-06T15:26:30.111+0000] {logging_mixin.py:188} INFO - Deleted unneeded file: /tmp/skills/.git/hooks/applypatch-msg.sample
[2025-04-06T15:26:30.112+0000] {logging_mixin.py:188} INFO - Deleted unneeded file: /tmp/skills/.git/hooks/pre-push.sample
[2025-04-06T15:26:30.113+0000] {logging_mixin.py:188} INFO - Deleted unneeded file: /tmp/skills/.git/hooks/pre-commit.sample
[2025-04-06T15:26:30.113+0000] {logging_mixin.py:188} INFO - Deleted unneeded file: /tmp/skills/.git/info/exclude
[2025-04-06T15:26:30.114+0000] {logging_mixin.py:188} WARNING - Processing skills: 21it [00:00, 481.13it/s]
[2025-04-06T15:26:30.117+0000] {logging_mixin.py:188} INFO - Deleted temporary folder: /tmp/skills
[2025-04-06T15:26:30.931+0000] {logging_mixin.py:188} WARNING - Processing CS50x-Coursework-Projects: 0it [00:00, ?it/s]
[2025-04-06T15:26:30.933+0000] {logging_mixin.py:188} INFO - Deleted unneeded file: /tmp/CS50x-Coursework-Projects/README.md
[2025-04-06T15:26:30.935+0000] {logging_mixin.py:188} INFO - Deleted unneeded file: /tmp/CS50x-Coursework-Projects/Week 4 Memory/README.md
[2025-04-06T15:26:30.940+0000] {logging_mixin.py:188} INFO - Deleted unneeded file: /tmp/CS50x-Coursework-Projects/Week 4 Memory/Recover/recover.c
[2025-04-06T15:26:30.944+0000] {logging_mixin.py:188} INFO - Deleted unneeded file: /tmp/CS50x-Coursework-Projects/Week 4 Memory/Recover/README (13).md
[2025-04-06T15:26:30.946+0000] {logging_mixin.py:188} INFO - Deleted unneeded file: /tmp/CS50x-Coursework-Projects/Week 4 Memory/Filter/helpers.c
[2025-04-06T15:26:30.947+0000] {logging_mixin.py:188} INFO - Deleted unneeded file: /tmp/CS50x-Coursework-Projects/Week 4 Memory/Filter/README (12).md
[2025-04-06T15:26:30.948+0000] {logging_mixin.py:188} INFO - Deleted unneeded file: /tmp/CS50x-Coursework-Projects/Week 4 Memory/Volume/README.md
[2025-04-06T15:26:30.949+0000] {logging_mixin.py:188} INFO - Deleted unneeded file: /tmp/CS50x-Coursework-Projects/Week 4 Memory/Volume/volume.c
[2025-04-06T15:26:30.949+0000] {logging_mixin.py:188} INFO - Deleted unneeded file: /tmp/CS50x-Coursework-Projects/Week 6 Python/README.md
[2025-04-06T15:26:30.950+0000] {logging_mixin.py:188} INFO - Deleted unneeded file: /tmp/CS50x-Coursework-Projects/Week 6 Python/Mario/README.md
[2025-04-06T15:26:30.954+0000] {_metadata.py:139} WARNING - Compute Engine Metadata server unavailable on attempt 1 of 3. Reason: [Errno 111] Connection refused
[2025-04-06T15:26:30.959+0000] {_metadata.py:139} WARNING - Compute Engine Metadata server unavailable on attempt 2 of 3. Reason: [Errno 111] Connection refused
[2025-04-06T15:26:30.963+0000] {_metadata.py:139} WARNING - Compute Engine Metadata server unavailable on attempt 3 of 3. Reason: [Errno 111] Connection refused
[2025-04-06T15:26:30.964+0000] {_default.py:338} WARNING - Authentication failed using Compute Engine authentication due to unavailable metadata server.
[2025-04-06T15:26:30.965+0000] {logging_mixin.py:188} INFO - Error initializing GCS client: Your default credentials were not found. To set up Application Default Credentials, see https://cloud.google.com/docs/authentication/external/set-up-adc for more information.
[2025-04-06T15:26:30.966+0000] {logging_mixin.py:188} INFO - Deleted unneeded file: /tmp/CS50x-Coursework-Projects/Week 6 Python/Hello/README.md
[2025-04-06T15:26:30.970+0000] {_metadata.py:139} WARNING - Compute Engine Metadata server unavailable on attempt 1 of 3. Reason: [Errno 111] Connection refused
[2025-04-06T15:26:30.974+0000] {_metadata.py:139} WARNING - Compute Engine Metadata server unavailable on attempt 2 of 3. Reason: [Errno 111] Connection refused
[2025-04-06T15:26:30.978+0000] {_metadata.py:139} WARNING - Compute Engine Metadata server unavailable on attempt 3 of 3. Reason: [Errno 111] Connection refused
[2025-04-06T15:26:30.979+0000] {_default.py:338} WARNING - Authentication failed using Compute Engine authentication due to unavailable metadata server.
[2025-04-06T15:26:30.980+0000] {logging_mixin.py:188} INFO - Error initializing GCS client: Your default credentials were not found. To set up Application Default Credentials, see https://cloud.google.com/docs/authentication/external/set-up-adc for more information.
[2025-04-06T15:26:30.998+0000] {_metadata.py:139} WARNING - Compute Engine Metadata server unavailable on attempt 1 of 3. Reason: [Errno 111] Connection refused
[2025-04-06T15:26:31.004+0000] {_metadata.py:139} WARNING - Compute Engine Metadata server unavailable on attempt 2 of 3. Reason: [Errno 111] Connection refused
[2025-04-06T15:26:31.007+0000] {_metadata.py:139} WARNING - Compute Engine Metadata server unavailable on attempt 3 of 3. Reason: [Errno 111] Connection refused
[2025-04-06T15:26:31.008+0000] {_default.py:338} WARNING - Authentication failed using Compute Engine authentication due to unavailable metadata server.
[2025-04-06T15:26:31.009+0000] {logging_mixin.py:188} INFO - Error initializing GCS client: Your default credentials were not found. To set up Application Default Credentials, see https://cloud.google.com/docs/authentication/external/set-up-adc for more information.
[2025-04-06T15:26:31.011+0000] {logging_mixin.py:188} INFO - Deleted unneeded file: /tmp/CS50x-Coursework-Projects/Week 6 Python/Readability/README.md
[2025-04-06T15:26:31.012+0000] {logging_mixin.py:188} INFO - Deleted unneeded file: /tmp/CS50x-Coursework-Projects/Week 6 Python/Cash/README.md
[2025-04-06T15:26:31.015+0000] {_metadata.py:139} WARNING - Compute Engine Metadata server unavailable on attempt 1 of 3. Reason: [Errno 111] Connection refused
[2025-04-06T15:26:31.018+0000] {_metadata.py:139} WARNING - Compute Engine Metadata server unavailable on attempt 2 of 3. Reason: [Errno 111] Connection refused
[2025-04-06T15:26:31.021+0000] {_metadata.py:139} WARNING - Compute Engine Metadata server unavailable on attempt 3 of 3. Reason: [Errno 111] Connection refused
[2025-04-06T15:26:31.023+0000] {_default.py:338} WARNING - Authentication failed using Compute Engine authentication due to unavailable metadata server.
[2025-04-06T15:26:31.024+0000] {logging_mixin.py:188} INFO - Error initializing GCS client: Your default credentials were not found. To set up Application Default Credentials, see https://cloud.google.com/docs/authentication/external/set-up-adc for more information.
[2025-04-06T15:26:31.025+0000] {logging_mixin.py:188} INFO - Deleted unneeded file: /tmp/CS50x-Coursework-Projects/Week 6 Python/DNA/README.md
[2025-04-06T15:26:31.030+0000] {_metadata.py:139} WARNING - Compute Engine Metadata server unavailable on attempt 1 of 3. Reason: [Errno 111] Connection refused
[2025-04-06T15:26:31.034+0000] {_metadata.py:139} WARNING - Compute Engine Metadata server unavailable on attempt 2 of 3. Reason: [Errno 111] Connection refused
[2025-04-06T15:26:31.039+0000] {_metadata.py:139} WARNING - Compute Engine Metadata server unavailable on attempt 3 of 3. Reason: [Errno 111] Connection refused
[2025-04-06T15:26:31.040+0000] {_default.py:338} WARNING - Authentication failed using Compute Engine authentication due to unavailable metadata server.
[2025-04-06T15:26:31.041+0000] {logging_mixin.py:188} INFO - Error initializing GCS client: Your default credentials were not found. To set up Application Default Credentials, see https://cloud.google.com/docs/authentication/external/set-up-adc for more information.
[2025-04-06T15:26:31.041+0000] {logging_mixin.py:188} WARNING - Processing CS50x-Coursework-Projects: 12it [00:00, 110.06it/s]
[2025-04-06T15:26:31.043+0000] {logging_mixin.py:188} INFO - Deleted unneeded file: /tmp/CS50x-Coursework-Projects/Week 9 Flask/README.md
[2025-04-06T15:26:31.044+0000] {logging_mixin.py:188} INFO - Deleted unneeded file: /tmp/CS50x-Coursework-Projects/Week 9 Flask/Birthdays/README.md
[2025-04-06T15:26:31.046+0000] {logging_mixin.py:188} INFO - Deleted unneeded file: /tmp/CS50x-Coursework-Projects/Week 9 Flask/Birthdays/birthdays.db
[2025-04-06T15:26:31.049+0000] {_metadata.py:139} WARNING - Compute Engine Metadata server unavailable on attempt 1 of 3. Reason: [Errno 111] Connection refused
[2025-04-06T15:26:31.052+0000] {_metadata.py:139} WARNING - Compute Engine Metadata server unavailable on attempt 2 of 3. Reason: [Errno 111] Connection refused
[2025-04-06T15:26:31.056+0000] {_metadata.py:139} WARNING - Compute Engine Metadata server unavailable on attempt 3 of 3. Reason: [Errno 111] Connection refused
[2025-04-06T15:26:31.058+0000] {_default.py:338} WARNING - Authentication failed using Compute Engine authentication due to unavailable metadata server.
[2025-04-06T15:26:31.059+0000] {logging_mixin.py:188} INFO - Error initializing GCS client: Your default credentials were not found. To set up Application Default Credentials, see https://cloud.google.com/docs/authentication/external/set-up-adc for more information.
[2025-04-06T15:26:31.063+0000] {_metadata.py:139} WARNING - Compute Engine Metadata server unavailable on attempt 1 of 3. Reason: [Errno 111] Connection refused
[2025-04-06T15:26:31.066+0000] {_metadata.py:139} WARNING - Compute Engine Metadata server unavailable on attempt 2 of 3. Reason: [Errno 111] Connection refused
[2025-04-06T15:26:31.074+0000] {_metadata.py:139} WARNING - Compute Engine Metadata server unavailable on attempt 3 of 3. Reason: [Errno 111] Connection refused
[2025-04-06T15:26:31.075+0000] {_default.py:338} WARNING - Authentication failed using Compute Engine authentication due to unavailable metadata server.
[2025-04-06T15:26:31.076+0000] {logging_mixin.py:188} INFO - Error initializing GCS client: Your default credentials were not found. To set up Application Default Credentials, see https://cloud.google.com/docs/authentication/external/set-up-adc for more information.
[2025-04-06T15:26:31.080+0000] {_metadata.py:139} WARNING - Compute Engine Metadata server unavailable on attempt 1 of 3. Reason: [Errno 111] Connection refused
[2025-04-06T15:26:31.084+0000] {_metadata.py:139} WARNING - Compute Engine Metadata server unavailable on attempt 2 of 3. Reason: [Errno 111] Connection refused
[2025-04-06T15:26:31.091+0000] {_metadata.py:139} WARNING - Compute Engine Metadata server unavailable on attempt 3 of 3. Reason: [Errno 111] Connection refused
[2025-04-06T15:26:31.092+0000] {_default.py:338} WARNING - Authentication failed using Compute Engine authentication due to unavailable metadata server.
[2025-04-06T15:26:31.093+0000] {logging_mixin.py:188} INFO - Error initializing GCS client: Your default credentials were not found. To set up Application Default Credentials, see https://cloud.google.com/docs/authentication/external/set-up-adc for more information.
[2025-04-06T15:26:31.094+0000] {logging_mixin.py:188} INFO - Deleted unneeded file: /tmp/CS50x-Coursework-Projects/Week 9 Flask/Finance/README.md
[2025-04-06T15:26:31.098+0000] {_metadata.py:139} WARNING - Compute Engine Metadata server unavailable on attempt 1 of 3. Reason: [Errno 111] Connection refused
[2025-04-06T15:26:31.106+0000] {_metadata.py:139} WARNING - Compute Engine Metadata server unavailable on attempt 2 of 3. Reason: [Errno 111] Connection refused
[2025-04-06T15:26:31.111+0000] {_metadata.py:139} WARNING - Compute Engine Metadata server unavailable on attempt 3 of 3. Reason: [Errno 111] Connection refused
[2025-04-06T15:26:31.112+0000] {_default.py:338} WARNING - Authentication failed using Compute Engine authentication due to unavailable metadata server.
[2025-04-06T15:26:31.114+0000] {logging_mixin.py:188} INFO - Error initializing GCS client: Your default credentials were not found. To set up Application Default Credentials, see https://cloud.google.com/docs/authentication/external/set-up-adc for more information.
[2025-04-06T15:26:31.117+0000] {_metadata.py:139} WARNING - Compute Engine Metadata server unavailable on attempt 1 of 3. Reason: [Errno 111] Connection refused
[2025-04-06T15:26:31.120+0000] {_metadata.py:139} WARNING - Compute Engine Metadata server unavailable on attempt 2 of 3. Reason: [Errno 111] Connection refused
[2025-04-06T15:26:31.128+0000] {_metadata.py:139} WARNING - Compute Engine Metadata server unavailable on attempt 3 of 3. Reason: [Errno 111] Connection refused
[2025-04-06T15:26:31.129+0000] {_default.py:338} WARNING - Authentication failed using Compute Engine authentication due to unavailable metadata server.
[2025-04-06T15:26:31.130+0000] {logging_mixin.py:188} INFO - Error initializing GCS client: Your default credentials were not found. To set up Application Default Credentials, see https://cloud.google.com/docs/authentication/external/set-up-adc for more information.
[2025-04-06T15:26:31.133+0000] {_metadata.py:139} WARNING - Compute Engine Metadata server unavailable on attempt 1 of 3. Reason: [Errno 111] Connection refused
[2025-04-06T15:26:31.137+0000] {_metadata.py:139} WARNING - Compute Engine Metadata server unavailable on attempt 2 of 3. Reason: [Errno 111] Connection refused
[2025-04-06T15:26:31.144+0000] {_metadata.py:139} WARNING - Compute Engine Metadata server unavailable on attempt 3 of 3. Reason: [Errno 111] Connection refused
[2025-04-06T15:26:31.145+0000] {_default.py:338} WARNING - Authentication failed using Compute Engine authentication due to unavailable metadata server.
[2025-04-06T15:26:31.146+0000] {logging_mixin.py:188} INFO - Error initializing GCS client: Your default credentials were not found. To set up Application Default Credentials, see https://cloud.google.com/docs/authentication/external/set-up-adc for more information.
[2025-04-06T15:26:31.147+0000] {logging_mixin.py:188} INFO - Deleted unneeded file: /tmp/CS50x-Coursework-Projects/Week 9 Flask/Finance/requirements.txt
[2025-04-06T15:26:31.148+0000] {logging_mixin.py:188} INFO - Deleted unneeded file: /tmp/CS50x-Coursework-Projects/Week 9 Flask/Finance/schema.sql
[2025-04-06T15:26:31.151+0000] {_metadata.py:139} WARNING - Compute Engine Metadata server unavailable on attempt 1 of 3. Reason: [Errno 111] Connection refused
[2025-04-06T15:26:31.154+0000] {_metadata.py:139} WARNING - Compute Engine Metadata server unavailable on attempt 2 of 3. Reason: [Errno 111] Connection refused
[2025-04-06T15:26:31.168+0000] {_metadata.py:139} WARNING - Compute Engine Metadata server unavailable on attempt 3 of 3. Reason: [Errno 111] Connection refused
[2025-04-06T15:26:31.179+0000] {_default.py:338} WARNING - Authentication failed using Compute Engine authentication due to unavailable metadata server.
[2025-04-06T15:26:31.180+0000] {logging_mixin.py:188} INFO - Error initializing GCS client: Your default credentials were not found. To set up Application Default Credentials, see https://cloud.google.com/docs/authentication/external/set-up-adc for more information.
[2025-04-06T15:26:31.181+0000] {logging_mixin.py:188} INFO - Deleted unneeded file: /tmp/CS50x-Coursework-Projects/Week 9 Flask/Finance/trades_create.sql
[2025-04-06T15:26:31.182+0000] {logging_mixin.py:188} INFO - Deleted unneeded file: /tmp/CS50x-Coursework-Projects/Week 9 Flask/Finance/finance.db
[2025-04-06T15:26:31.183+0000] {logging_mixin.py:188} INFO - Deleted unneeded file: /tmp/CS50x-Coursework-Projects/Week 9 Flask/Finance/static/favicon.ico
[2025-04-06T15:26:31.189+0000] {_metadata.py:139} WARNING - Compute Engine Metadata server unavailable on attempt 1 of 3. Reason: [Errno 111] Connection refused
[2025-04-06T15:26:31.192+0000] {_metadata.py:139} WARNING - Compute Engine Metadata server unavailable on attempt 2 of 3. Reason: [Errno 111] Connection refused
[2025-04-06T15:26:31.198+0000] {_metadata.py:139} WARNING - Compute Engine Metadata server unavailable on attempt 3 of 3. Reason: [Errno 111] Connection refused
[2025-04-06T15:26:31.201+0000] {_default.py:338} WARNING - Authentication failed using Compute Engine authentication due to unavailable metadata server.
[2025-04-06T15:26:31.202+0000] {logging_mixin.py:188} INFO - Error initializing GCS client: Your default credentials were not found. To set up Application Default Credentials, see https://cloud.google.com/docs/authentication/external/set-up-adc for more information.
[2025-04-06T15:26:31.206+0000] {_metadata.py:139} WARNING - Compute Engine Metadata server unavailable on attempt 1 of 3. Reason: [Errno 111] Connection refused
[2025-04-06T15:26:31.210+0000] {_metadata.py:139} WARNING - Compute Engine Metadata server unavailable on attempt 2 of 3. Reason: [Errno 111] Connection refused
[2025-04-06T15:26:31.213+0000] {_metadata.py:139} WARNING - Compute Engine Metadata server unavailable on attempt 3 of 3. Reason: [Errno 111] Connection refused
[2025-04-06T15:26:31.215+0000] {_default.py:338} WARNING - Authentication failed using Compute Engine authentication due to unavailable metadata server.
[2025-04-06T15:26:31.215+0000] {logging_mixin.py:188} INFO - Error initializing GCS client: Your default credentials were not found. To set up Application Default Credentials, see https://cloud.google.com/docs/authentication/external/set-up-adc for more information.
[2025-04-06T15:26:31.234+0000] {_metadata.py:139} WARNING - Compute Engine Metadata server unavailable on attempt 1 of 3. Reason: [Errno 111] Connection refused
[2025-04-06T15:26:31.239+0000] {_metadata.py:139} WARNING - Compute Engine Metadata server unavailable on attempt 2 of 3. Reason: [Errno 111] Connection refused
[2025-04-06T15:26:31.242+0000] {_metadata.py:139} WARNING - Compute Engine Metadata server unavailable on attempt 3 of 3. Reason: [Errno 111] Connection refused
[2025-04-06T15:26:31.244+0000] {_default.py:338} WARNING - Authentication failed using Compute Engine authentication due to unavailable metadata server.
[2025-04-06T15:26:31.245+0000] {logging_mixin.py:188} INFO - Error initializing GCS client: Your default credentials were not found. To set up Application Default Credentials, see https://cloud.google.com/docs/authentication/external/set-up-adc for more information.
[2025-04-06T15:26:31.250+0000] {_metadata.py:139} WARNING - Compute Engine Metadata server unavailable on attempt 1 of 3. Reason: [Errno 111] Connection refused
[2025-04-06T15:26:31.253+0000] {_metadata.py:139} WARNING - Compute Engine Metadata server unavailable on attempt 2 of 3. Reason: [Errno 111] Connection refused
[2025-04-06T15:26:31.257+0000] {_metadata.py:139} WARNING - Compute Engine Metadata server unavailable on attempt 3 of 3. Reason: [Errno 111] Connection refused
[2025-04-06T15:26:31.258+0000] {_default.py:338} WARNING - Authentication failed using Compute Engine authentication due to unavailable metadata server.
[2025-04-06T15:26:31.259+0000] {logging_mixin.py:188} INFO - Error initializing GCS client: Your default credentials were not found. To set up Application Default Credentials, see https://cloud.google.com/docs/authentication/external/set-up-adc for more information.
[2025-04-06T15:26:31.262+0000] {_metadata.py:139} WARNING - Compute Engine Metadata server unavailable on attempt 1 of 3. Reason: [Errno 111] Connection refused
[2025-04-06T15:26:31.267+0000] {_metadata.py:139} WARNING - Compute Engine Metadata server unavailable on attempt 2 of 3. Reason: [Errno 111] Connection refused
[2025-04-06T15:26:31.271+0000] {_metadata.py:139} WARNING - Compute Engine Metadata server unavailable on attempt 3 of 3. Reason: [Errno 111] Connection refused
[2025-04-06T15:26:31.272+0000] {_default.py:338} WARNING - Authentication failed using Compute Engine authentication due to unavailable metadata server.
[2025-04-06T15:26:31.273+0000] {logging_mixin.py:188} INFO - Error initializing GCS client: Your default credentials were not found. To set up Application Default Credentials, see https://cloud.google.com/docs/authentication/external/set-up-adc for more information.
[2025-04-06T15:26:31.278+0000] {_metadata.py:139} WARNING - Compute Engine Metadata server unavailable on attempt 1 of 3. Reason: [Errno 111] Connection refused
[2025-04-06T15:26:31.281+0000] {_metadata.py:139} WARNING - Compute Engine Metadata server unavailable on attempt 2 of 3. Reason: [Errno 111] Connection refused
[2025-04-06T15:26:31.285+0000] {_metadata.py:139} WARNING - Compute Engine Metadata server unavailable on attempt 3 of 3. Reason: [Errno 111] Connection refused
[2025-04-06T15:26:31.286+0000] {_default.py:338} WARNING - Authentication failed using Compute Engine authentication due to unavailable metadata server.
[2025-04-06T15:26:31.287+0000] {logging_mixin.py:188} INFO - Error initializing GCS client: Your default credentials were not found. To set up Application Default Credentials, see https://cloud.google.com/docs/authentication/external/set-up-adc for more information.
[2025-04-06T15:26:31.289+0000] {_metadata.py:139} WARNING - Compute Engine Metadata server unavailable on attempt 1 of 3. Reason: [Errno 111] Connection refused
[2025-04-06T15:26:31.292+0000] {_metadata.py:139} WARNING - Compute Engine Metadata server unavailable on attempt 2 of 3. Reason: [Errno 111] Connection refused
[2025-04-06T15:26:31.295+0000] {_metadata.py:139} WARNING - Compute Engine Metadata server unavailable on attempt 3 of 3. Reason: [Errno 111] Connection refused
[2025-04-06T15:26:31.296+0000] {_default.py:338} WARNING - Authentication failed using Compute Engine authentication due to unavailable metadata server.
[2025-04-06T15:26:31.297+0000] {logging_mixin.py:188} INFO - Error initializing GCS client: Your default credentials were not found. To set up Application Default Credentials, see https://cloud.google.com/docs/authentication/external/set-up-adc for more information.
[2025-04-06T15:26:31.301+0000] {_metadata.py:139} WARNING - Compute Engine Metadata server unavailable on attempt 1 of 3. Reason: [Errno 111] Connection refused
[2025-04-06T15:26:31.304+0000] {_metadata.py:139} WARNING - Compute Engine Metadata server unavailable on attempt 2 of 3. Reason: [Errno 111] Connection refused
[2025-04-06T15:26:31.308+0000] {_metadata.py:139} WARNING - Compute Engine Metadata server unavailable on attempt 3 of 3. Reason: [Errno 111] Connection refused
[2025-04-06T15:26:31.309+0000] {_default.py:338} WARNING - Authentication failed using Compute Engine authentication due to unavailable metadata server.
[2025-04-06T15:26:31.310+0000] {logging_mixin.py:188} INFO - Error initializing GCS client: Your default credentials were not found. To set up Application Default Credentials, see https://cloud.google.com/docs/authentication/external/set-up-adc for more information.
[2025-04-06T15:26:31.313+0000] {_metadata.py:139} WARNING - Compute Engine Metadata server unavailable on attempt 1 of 3. Reason: [Errno 111] Connection refused
[2025-04-06T15:26:31.316+0000] {_metadata.py:139} WARNING - Compute Engine Metadata server unavailable on attempt 2 of 3. Reason: [Errno 111] Connection refused
[2025-04-06T15:26:31.319+0000] {_metadata.py:139} WARNING - Compute Engine Metadata server unavailable on attempt 3 of 3. Reason: [Errno 111] Connection refused
[2025-04-06T15:26:31.320+0000] {_default.py:338} WARNING - Authentication failed using Compute Engine authentication due to unavailable metadata server.
[2025-04-06T15:26:31.321+0000] {logging_mixin.py:188} INFO - Error initializing GCS client: Your default credentials were not found. To set up Application Default Credentials, see https://cloud.google.com/docs/authentication/external/set-up-adc for more information.
[2025-04-06T15:26:31.325+0000] {_metadata.py:139} WARNING - Compute Engine Metadata server unavailable on attempt 1 of 3. Reason: [Errno 111] Connection refused
[2025-04-06T15:26:31.329+0000] {_metadata.py:139} WARNING - Compute Engine Metadata server unavailable on attempt 2 of 3. Reason: [Errno 111] Connection refused
[2025-04-06T15:26:31.332+0000] {_metadata.py:139} WARNING - Compute Engine Metadata server unavailable on attempt 3 of 3. Reason: [Errno 111] Connection refused
[2025-04-06T15:26:31.333+0000] {_default.py:338} WARNING - Authentication failed using Compute Engine authentication due to unavailable metadata server.
[2025-04-06T15:26:31.333+0000] {logging_mixin.py:188} INFO - Error initializing GCS client: Your default credentials were not found. To set up Application Default Credentials, see https://cloud.google.com/docs/authentication/external/set-up-adc for more information.
[2025-04-06T15:26:31.337+0000] {_metadata.py:139} WARNING - Compute Engine Metadata server unavailable on attempt 1 of 3. Reason: [Errno 111] Connection refused
[2025-04-06T15:26:31.340+0000] {_metadata.py:139} WARNING - Compute Engine Metadata server unavailable on attempt 2 of 3. Reason: [Errno 111] Connection refused
[2025-04-06T15:26:31.344+0000] {_metadata.py:139} WARNING - Compute Engine Metadata server unavailable on attempt 3 of 3. Reason: [Errno 111] Connection refused
[2025-04-06T15:26:31.345+0000] {_default.py:338} WARNING - Authentication failed using Compute Engine authentication due to unavailable metadata server.
[2025-04-06T15:26:31.345+0000] {logging_mixin.py:188} INFO - Error initializing GCS client: Your default credentials were not found. To set up Application Default Credentials, see https://cloud.google.com/docs/authentication/external/set-up-adc for more information.
[2025-04-06T15:26:31.350+0000] {_metadata.py:139} WARNING - Compute Engine Metadata server unavailable on attempt 1 of 3. Reason: [Errno 111] Connection refused
[2025-04-06T15:26:31.353+0000] {_metadata.py:139} WARNING - Compute Engine Metadata server unavailable on attempt 2 of 3. Reason: [Errno 111] Connection refused
[2025-04-06T15:26:31.357+0000] {_metadata.py:139} WARNING - Compute Engine Metadata server unavailable on attempt 3 of 3. Reason: [Errno 111] Connection refused
[2025-04-06T15:26:31.358+0000] {_default.py:338} WARNING - Authentication failed using Compute Engine authentication due to unavailable metadata server.
[2025-04-06T15:26:31.359+0000] {logging_mixin.py:188} INFO - Error initializing GCS client: Your default credentials were not found. To set up Application Default Credentials, see https://cloud.google.com/docs/authentication/external/set-up-adc for more information.
[2025-04-06T15:26:31.363+0000] {_metadata.py:139} WARNING - Compute Engine Metadata server unavailable on attempt 1 of 3. Reason: [Errno 111] Connection refused
[2025-04-06T15:26:31.369+0000] {_metadata.py:139} WARNING - Compute Engine Metadata server unavailable on attempt 2 of 3. Reason: [Errno 111] Connection refused
[2025-04-06T15:26:31.382+0000] {_metadata.py:139} WARNING - Compute Engine Metadata server unavailable on attempt 3 of 3. Reason: [Errno 111] Connection refused
[2025-04-06T15:26:31.383+0000] {_default.py:338} WARNING - Authentication failed using Compute Engine authentication due to unavailable metadata server.
[2025-04-06T15:26:31.384+0000] {logging_mixin.py:188} INFO - Error initializing GCS client: Your default credentials were not found. To set up Application Default Credentials, see https://cloud.google.com/docs/authentication/external/set-up-adc for more information.
[2025-04-06T15:26:31.385+0000] {logging_mixin.py:188} INFO - Deleted unneeded file: /tmp/CS50x-Coursework-Projects/Week 7 SQL/README.md
[2025-04-06T15:26:31.386+0000] {logging_mixin.py:188} INFO - Deleted unneeded file: /tmp/CS50x-Coursework-Projects/Week 7 SQL/Fiftyville/README.md
[2025-04-06T15:26:31.387+0000] {logging_mixin.py:188} INFO - Deleted unneeded file: /tmp/CS50x-Coursework-Projects/Week 7 SQL/Fiftyville/answers.txt
[2025-04-06T15:26:31.388+0000] {logging_mixin.py:188} INFO - Deleted unneeded file: /tmp/CS50x-Coursework-Projects/Week 7 SQL/Fiftyville/log.sql
[2025-04-06T15:26:31.389+0000] {logging_mixin.py:188} INFO - Deleted unneeded file: /tmp/CS50x-Coursework-Projects/Week 7 SQL/Movies/11.sql
[2025-04-06T15:26:31.390+0000] {logging_mixin.py:188} INFO - Deleted unneeded file: /tmp/CS50x-Coursework-Projects/Week 7 SQL/Movies/3.sql
[2025-04-06T15:26:31.391+0000] {logging_mixin.py:188} INFO - Deleted unneeded file: /tmp/CS50x-Coursework-Projects/Week 7 SQL/Movies/4.sql
[2025-04-06T15:26:31.392+0000] {logging_mixin.py:188} INFO - Deleted unneeded file: /tmp/CS50x-Coursework-Projects/Week 7 SQL/Movies/1.sql
[2025-04-06T15:26:31.393+0000] {logging_mixin.py:188} INFO - Deleted unneeded file: /tmp/CS50x-Coursework-Projects/Week 7 SQL/Movies/7.sql
[2025-04-06T15:26:31.394+0000] {logging_mixin.py:188} INFO - Deleted unneeded file: /tmp/CS50x-Coursework-Projects/Week 7 SQL/Movies/README.md
[2025-04-06T15:26:31.394+0000] {logging_mixin.py:188} INFO - Deleted unneeded file: /tmp/CS50x-Coursework-Projects/Week 7 SQL/Movies/8.sql
[2025-04-06T15:26:31.396+0000] {logging_mixin.py:188} INFO - Deleted unneeded file: /tmp/CS50x-Coursework-Projects/Week 7 SQL/Movies/9.sql
[2025-04-06T15:26:31.397+0000] {logging_mixin.py:188} INFO - Deleted unneeded file: /tmp/CS50x-Coursework-Projects/Week 7 SQL/Movies/2.sql
[2025-04-06T15:26:31.398+0000] {logging_mixin.py:188} INFO - Deleted unneeded file: /tmp/CS50x-Coursework-Projects/Week 7 SQL/Movies/10.sql
[2025-04-06T15:26:31.399+0000] {logging_mixin.py:188} INFO - Deleted unneeded file: /tmp/CS50x-Coursework-Projects/Week 7 SQL/Movies/6.sql
[2025-04-06T15:26:31.400+0000] {logging_mixin.py:188} INFO - Deleted unneeded file: /tmp/CS50x-Coursework-Projects/Week 7 SQL/Movies/13.sql
[2025-04-06T15:26:31.401+0000] {logging_mixin.py:188} INFO - Deleted unneeded file: /tmp/CS50x-Coursework-Projects/Week 7 SQL/Movies/12.sql
[2025-04-06T15:26:31.402+0000] {logging_mixin.py:188} INFO - Deleted unneeded file: /tmp/CS50x-Coursework-Projects/Week 7 SQL/Movies/5.sql
[2025-04-06T15:26:31.404+0000] {logging_mixin.py:188} INFO - Deleted unneeded file: /tmp/CS50x-Coursework-Projects/Week 7 SQL/Songs/3.sql
[2025-04-06T15:26:31.404+0000] {logging_mixin.py:188} INFO - Deleted unneeded file: /tmp/CS50x-Coursework-Projects/Week 7 SQL/Songs/4.sql
[2025-04-06T15:26:31.405+0000] {logging_mixin.py:188} INFO - Deleted unneeded file: /tmp/CS50x-Coursework-Projects/Week 7 SQL/Songs/1.sql
[2025-04-06T15:26:31.406+0000] {logging_mixin.py:188} INFO - Deleted unneeded file: /tmp/CS50x-Coursework-Projects/Week 7 SQL/Songs/7.sql
[2025-04-06T15:26:31.407+0000] {logging_mixin.py:188} INFO - Deleted unneeded file: /tmp/CS50x-Coursework-Projects/Week 7 SQL/Songs/README.md
[2025-04-06T15:26:31.408+0000] {logging_mixin.py:188} INFO - Deleted unneeded file: /tmp/CS50x-Coursework-Projects/Week 7 SQL/Songs/8.sql
[2025-04-06T15:26:31.408+0000] {logging_mixin.py:188} INFO - Deleted unneeded file: /tmp/CS50x-Coursework-Projects/Week 7 SQL/Songs/answers.txt
[2025-04-06T15:26:31.409+0000] {logging_mixin.py:188} INFO - Deleted unneeded file: /tmp/CS50x-Coursework-Projects/Week 7 SQL/Songs/2.sql
[2025-04-06T15:26:31.410+0000] {logging_mixin.py:188} INFO - Deleted unneeded file: /tmp/CS50x-Coursework-Projects/Week 7 SQL/Songs/6.sql
[2025-04-06T15:26:31.410+0000] {logging_mixin.py:188} INFO - Deleted unneeded file: /tmp/CS50x-Coursework-Projects/Week 7 SQL/Songs/5.sql
[2025-04-06T15:26:31.411+0000] {logging_mixin.py:188} INFO - Deleted unneeded file: /tmp/CS50x-Coursework-Projects/Week 2 Arrays/README.md
[2025-04-06T15:26:31.412+0000] {logging_mixin.py:188} WARNING - Processing CS50x-Coursework-Projects: 24it [00:00, 45.62it/s] 
[2025-04-06T15:26:31.414+0000] {logging_mixin.py:188} INFO - Deleted unneeded file: /tmp/CS50x-Coursework-Projects/Week 2 Arrays/Readability/readability
[2025-04-06T15:26:31.415+0000] {logging_mixin.py:188} INFO - Deleted unneeded file: /tmp/CS50x-Coursework-Projects/Week 2 Arrays/Readability/readability.c
[2025-04-06T15:26:31.416+0000] {logging_mixin.py:188} INFO - Deleted unneeded file: /tmp/CS50x-Coursework-Projects/Week 2 Arrays/Readability/README (6).md
[2025-04-06T15:26:31.417+0000] {logging_mixin.py:188} INFO - Deleted unneeded file: /tmp/CS50x-Coursework-Projects/Week 2 Arrays/Scrabble/README.md
[2025-04-06T15:26:31.418+0000] {logging_mixin.py:188} INFO - Deleted unneeded file: /tmp/CS50x-Coursework-Projects/Week 2 Arrays/Scrabble/scrabble
[2025-04-06T15:26:31.419+0000] {logging_mixin.py:188} INFO - Deleted unneeded file: /tmp/CS50x-Coursework-Projects/Week 2 Arrays/Scrabble/scrabble.c
[2025-04-06T15:26:31.424+0000] {logging_mixin.py:188} INFO - Deleted unneeded file: /tmp/CS50x-Coursework-Projects/Week 2 Arrays/Caesar/caesar.c
[2025-04-06T15:26:31.426+0000] {logging_mixin.py:188} INFO - Deleted unneeded file: /tmp/CS50x-Coursework-Projects/Week 2 Arrays/Caesar/README (5).md
[2025-04-06T15:26:31.427+0000] {logging_mixin.py:188} INFO - Deleted unneeded file: /tmp/CS50x-Coursework-Projects/Week 2 Arrays/Caesar/caesar
[2025-04-06T15:26:31.428+0000] {logging_mixin.py:188} INFO - Deleted unneeded file: /tmp/CS50x-Coursework-Projects/Week 10 The End/Final Project/CDatabase.sql
[2025-04-06T15:26:31.429+0000] {logging_mixin.py:188} INFO - Deleted unneeded file: /tmp/CS50x-Coursework-Projects/Week 10 The End/Final Project/README.md
[2025-04-06T15:26:31.430+0000] {logging_mixin.py:188} INFO - Deleted unneeded file: /tmp/CS50x-Coursework-Projects/Week 10 The End/Final Project/run
[2025-04-06T15:26:31.431+0000] {logging_mixin.py:188} INFO - Deleted unneeded file: /tmp/CS50x-Coursework-Projects/Week 10 The End/Final Project/requirements.txt
[2025-04-06T15:26:31.434+0000] {_metadata.py:139} WARNING - Compute Engine Metadata server unavailable on attempt 1 of 3. Reason: [Errno 111] Connection refused
[2025-04-06T15:26:31.438+0000] {_metadata.py:139} WARNING - Compute Engine Metadata server unavailable on attempt 2 of 3. Reason: [Errno 111] Connection refused
[2025-04-06T15:26:31.441+0000] {_metadata.py:139} WARNING - Compute Engine Metadata server unavailable on attempt 3 of 3. Reason: [Errno 111] Connection refused
[2025-04-06T15:26:31.442+0000] {_default.py:338} WARNING - Authentication failed using Compute Engine authentication due to unavailable metadata server.
[2025-04-06T15:26:31.443+0000] {logging_mixin.py:188} INFO - Error initializing GCS client: Your default credentials were not found. To set up Application Default Credentials, see https://cloud.google.com/docs/authentication/external/set-up-adc for more information.
[2025-04-06T15:26:31.446+0000] {_metadata.py:139} WARNING - Compute Engine Metadata server unavailable on attempt 1 of 3. Reason: [Errno 111] Connection refused
[2025-04-06T15:26:31.449+0000] {_metadata.py:139} WARNING - Compute Engine Metadata server unavailable on attempt 2 of 3. Reason: [Errno 111] Connection refused
[2025-04-06T15:26:31.452+0000] {_metadata.py:139} WARNING - Compute Engine Metadata server unavailable on attempt 3 of 3. Reason: [Errno 111] Connection refused
[2025-04-06T15:26:31.453+0000] {_default.py:338} WARNING - Authentication failed using Compute Engine authentication due to unavailable metadata server.
[2025-04-06T15:26:31.453+0000] {logging_mixin.py:188} INFO - Error initializing GCS client: Your default credentials were not found. To set up Application Default Credentials, see https://cloud.google.com/docs/authentication/external/set-up-adc for more information.
[2025-04-06T15:26:31.456+0000] {_metadata.py:139} WARNING - Compute Engine Metadata server unavailable on attempt 1 of 3. Reason: [Errno 111] Connection refused
[2025-04-06T15:26:31.461+0000] {_metadata.py:139} WARNING - Compute Engine Metadata server unavailable on attempt 2 of 3. Reason: [Errno 111] Connection refused
[2025-04-06T15:26:31.464+0000] {_metadata.py:139} WARNING - Compute Engine Metadata server unavailable on attempt 3 of 3. Reason: [Errno 111] Connection refused
[2025-04-06T15:26:31.465+0000] {_default.py:338} WARNING - Authentication failed using Compute Engine authentication due to unavailable metadata server.
[2025-04-06T15:26:31.466+0000] {logging_mixin.py:188} INFO - Error initializing GCS client: Your default credentials were not found. To set up Application Default Credentials, see https://cloud.google.com/docs/authentication/external/set-up-adc for more information.
[2025-04-06T15:26:31.467+0000] {logging_mixin.py:188} INFO - Deleted unneeded file: /tmp/CS50x-Coursework-Projects/Week 10 The End/Final Project/__pycache__/main.cpython-36.pyc
[2025-04-06T15:26:31.468+0000] {logging_mixin.py:188} INFO - Deleted unneeded file: /tmp/CS50x-Coursework-Projects/Week 10 The End/Final Project/__pycache__/main.cpython-310.pyc
[2025-04-06T15:26:31.469+0000] {logging_mixin.py:188} INFO - Deleted unneeded file: /tmp/CS50x-Coursework-Projects/Week 10 The End/Final Project/__pycache__/main.cpython-39.pyc
[2025-04-06T15:26:31.470+0000] {logging_mixin.py:188} INFO - Deleted unneeded file: /tmp/CS50x-Coursework-Projects/Week 10 The End/Final Project/static/validate.js
[2025-04-06T15:26:31.473+0000] {_metadata.py:139} WARNING - Compute Engine Metadata server unavailable on attempt 1 of 3. Reason: [Errno 111] Connection refused
[2025-04-06T15:26:31.476+0000] {_metadata.py:139} WARNING - Compute Engine Metadata server unavailable on attempt 2 of 3. Reason: [Errno 111] Connection refused
[2025-04-06T15:26:31.480+0000] {_metadata.py:139} WARNING - Compute Engine Metadata server unavailable on attempt 3 of 3. Reason: [Errno 111] Connection refused
[2025-04-06T15:26:31.482+0000] {_default.py:338} WARNING - Authentication failed using Compute Engine authentication due to unavailable metadata server.
[2025-04-06T15:26:31.483+0000] {logging_mixin.py:188} INFO - Error initializing GCS client: Your default credentials were not found. To set up Application Default Credentials, see https://cloud.google.com/docs/authentication/external/set-up-adc for more information.
[2025-04-06T15:26:31.489+0000] {_metadata.py:139} WARNING - Compute Engine Metadata server unavailable on attempt 1 of 3. Reason: [Errno 111] Connection refused
[2025-04-06T15:26:31.492+0000] {_metadata.py:139} WARNING - Compute Engine Metadata server unavailable on attempt 2 of 3. Reason: [Errno 111] Connection refused
[2025-04-06T15:26:31.496+0000] {_metadata.py:139} WARNING - Compute Engine Metadata server unavailable on attempt 3 of 3. Reason: [Errno 111] Connection refused
[2025-04-06T15:26:31.498+0000] {_default.py:338} WARNING - Authentication failed using Compute Engine authentication due to unavailable metadata server.
[2025-04-06T15:26:31.499+0000] {logging_mixin.py:188} INFO - Error initializing GCS client: Your default credentials were not found. To set up Application Default Credentials, see https://cloud.google.com/docs/authentication/external/set-up-adc for more information.
[2025-04-06T15:26:31.503+0000] {_metadata.py:139} WARNING - Compute Engine Metadata server unavailable on attempt 1 of 3. Reason: [Errno 111] Connection refused
[2025-04-06T15:26:31.506+0000] {_metadata.py:139} WARNING - Compute Engine Metadata server unavailable on attempt 2 of 3. Reason: [Errno 111] Connection refused
[2025-04-06T15:26:31.510+0000] {_metadata.py:139} WARNING - Compute Engine Metadata server unavailable on attempt 3 of 3. Reason: [Errno 111] Connection refused
[2025-04-06T15:26:31.511+0000] {_default.py:338} WARNING - Authentication failed using Compute Engine authentication due to unavailable metadata server.
[2025-04-06T15:26:31.512+0000] {logging_mixin.py:188} INFO - Error initializing GCS client: Your default credentials were not found. To set up Application Default Credentials, see https://cloud.google.com/docs/authentication/external/set-up-adc for more information.
[2025-04-06T15:26:31.515+0000] {_metadata.py:139} WARNING - Compute Engine Metadata server unavailable on attempt 1 of 3. Reason: [Errno 111] Connection refused
[2025-04-06T15:26:31.519+0000] {_metadata.py:139} WARNING - Compute Engine Metadata server unavailable on attempt 2 of 3. Reason: [Errno 111] Connection refused
[2025-04-06T15:26:31.525+0000] {_metadata.py:139} WARNING - Compute Engine Metadata server unavailable on attempt 3 of 3. Reason: [Errno 111] Connection refused
[2025-04-06T15:26:31.527+0000] {_default.py:338} WARNING - Authentication failed using Compute Engine authentication due to unavailable metadata server.
[2025-04-06T15:26:31.528+0000] {logging_mixin.py:188} INFO - Error initializing GCS client: Your default credentials were not found. To set up Application Default Credentials, see https://cloud.google.com/docs/authentication/external/set-up-adc for more information.
[2025-04-06T15:26:31.530+0000] {logging_mixin.py:188} WARNING - Processing CS50x-Coursework-Projects: 34it [00:00, 56.38it/s]
[2025-04-06T15:26:31.534+0000] {_metadata.py:139} WARNING - Compute Engine Metadata server unavailable on attempt 1 of 3. Reason: [Errno 111] Connection refused
[2025-04-06T15:26:31.537+0000] {_metadata.py:139} WARNING - Compute Engine Metadata server unavailable on attempt 2 of 3. Reason: [Errno 111] Connection refused
[2025-04-06T15:26:31.540+0000] {_metadata.py:139} WARNING - Compute Engine Metadata server unavailable on attempt 3 of 3. Reason: [Errno 111] Connection refused
[2025-04-06T15:26:31.541+0000] {_default.py:338} WARNING - Authentication failed using Compute Engine authentication due to unavailable metadata server.
[2025-04-06T15:26:31.542+0000] {logging_mixin.py:188} INFO - Error initializing GCS client: Your default credentials were not found. To set up Application Default Credentials, see https://cloud.google.com/docs/authentication/external/set-up-adc for more information.
[2025-04-06T15:26:31.546+0000] {_metadata.py:139} WARNING - Compute Engine Metadata server unavailable on attempt 1 of 3. Reason: [Errno 111] Connection refused
[2025-04-06T15:26:31.550+0000] {_metadata.py:139} WARNING - Compute Engine Metadata server unavailable on attempt 2 of 3. Reason: [Errno 111] Connection refused
[2025-04-06T15:26:31.553+0000] {_metadata.py:139} WARNING - Compute Engine Metadata server unavailable on attempt 3 of 3. Reason: [Errno 111] Connection refused
[2025-04-06T15:26:31.553+0000] {_default.py:338} WARNING - Authentication failed using Compute Engine authentication due to unavailable metadata server.
[2025-04-06T15:26:31.554+0000] {logging_mixin.py:188} INFO - Error initializing GCS client: Your default credentials were not found. To set up Application Default Credentials, see https://cloud.google.com/docs/authentication/external/set-up-adc for more information.
[2025-04-06T15:26:31.557+0000] {_metadata.py:139} WARNING - Compute Engine Metadata server unavailable on attempt 1 of 3. Reason: [Errno 111] Connection refused
[2025-04-06T15:26:31.561+0000] {_metadata.py:139} WARNING - Compute Engine Metadata server unavailable on attempt 2 of 3. Reason: [Errno 111] Connection refused
[2025-04-06T15:26:31.565+0000] {_metadata.py:139} WARNING - Compute Engine Metadata server unavailable on attempt 3 of 3. Reason: [Errno 111] Connection refused
[2025-04-06T15:26:31.565+0000] {_default.py:338} WARNING - Authentication failed using Compute Engine authentication due to unavailable metadata server.
[2025-04-06T15:26:31.566+0000] {logging_mixin.py:188} INFO - Error initializing GCS client: Your default credentials were not found. To set up Application Default Credentials, see https://cloud.google.com/docs/authentication/external/set-up-adc for more information.
[2025-04-06T15:26:31.569+0000] {_metadata.py:139} WARNING - Compute Engine Metadata server unavailable on attempt 1 of 3. Reason: [Errno 111] Connection refused
[2025-04-06T15:26:31.572+0000] {_metadata.py:139} WARNING - Compute Engine Metadata server unavailable on attempt 2 of 3. Reason: [Errno 111] Connection refused
[2025-04-06T15:26:31.575+0000] {_metadata.py:139} WARNING - Compute Engine Metadata server unavailable on attempt 3 of 3. Reason: [Errno 111] Connection refused
[2025-04-06T15:26:31.576+0000] {_default.py:338} WARNING - Authentication failed using Compute Engine authentication due to unavailable metadata server.
[2025-04-06T15:26:31.577+0000] {logging_mixin.py:188} INFO - Error initializing GCS client: Your default credentials were not found. To set up Application Default Credentials, see https://cloud.google.com/docs/authentication/external/set-up-adc for more information.
[2025-04-06T15:26:31.578+0000] {logging_mixin.py:188} INFO - Deleted unneeded file: /tmp/CS50x-Coursework-Projects/Week 8 HTML, CSS, JavaScript/README.md
[2025-04-06T15:26:31.582+0000] {_metadata.py:139} WARNING - Compute Engine Metadata server unavailable on attempt 1 of 3. Reason: [Errno 111] Connection refused
[2025-04-06T15:26:31.587+0000] {_metadata.py:139} WARNING - Compute Engine Metadata server unavailable on attempt 2 of 3. Reason: [Errno 111] Connection refused
[2025-04-06T15:26:31.590+0000] {_metadata.py:139} WARNING - Compute Engine Metadata server unavailable on attempt 3 of 3. Reason: [Errno 111] Connection refused
[2025-04-06T15:26:31.591+0000] {_default.py:338} WARNING - Authentication failed using Compute Engine authentication due to unavailable metadata server.
[2025-04-06T15:26:31.591+0000] {logging_mixin.py:188} INFO - Error initializing GCS client: Your default credentials were not found. To set up Application Default Credentials, see https://cloud.google.com/docs/authentication/external/set-up-adc for more information.
[2025-04-06T15:26:31.592+0000] {logging_mixin.py:188} INFO - Deleted unneeded file: /tmp/CS50x-Coursework-Projects/Week 8 HTML, CSS, JavaScript/Trivia/README.md
[2025-04-06T15:26:31.595+0000] {_metadata.py:139} WARNING - Compute Engine Metadata server unavailable on attempt 1 of 3. Reason: [Errno 111] Connection refused
[2025-04-06T15:26:31.598+0000] {_metadata.py:139} WARNING - Compute Engine Metadata server unavailable on attempt 2 of 3. Reason: [Errno 111] Connection refused
[2025-04-06T15:26:31.601+0000] {_metadata.py:139} WARNING - Compute Engine Metadata server unavailable on attempt 3 of 3. Reason: [Errno 111] Connection refused
[2025-04-06T15:26:31.602+0000] {_default.py:338} WARNING - Authentication failed using Compute Engine authentication due to unavailable metadata server.
[2025-04-06T15:26:31.603+0000] {logging_mixin.py:188} INFO - Error initializing GCS client: Your default credentials were not found. To set up Application Default Credentials, see https://cloud.google.com/docs/authentication/external/set-up-adc for more information.
[2025-04-06T15:26:31.606+0000] {_metadata.py:139} WARNING - Compute Engine Metadata server unavailable on attempt 1 of 3. Reason: [Errno 111] Connection refused
[2025-04-06T15:26:31.610+0000] {_metadata.py:139} WARNING - Compute Engine Metadata server unavailable on attempt 2 of 3. Reason: [Errno 111] Connection refused
[2025-04-06T15:26:31.615+0000] {_metadata.py:139} WARNING - Compute Engine Metadata server unavailable on attempt 3 of 3. Reason: [Errno 111] Connection refused
[2025-04-06T15:26:31.616+0000] {_default.py:338} WARNING - Authentication failed using Compute Engine authentication due to unavailable metadata server.
[2025-04-06T15:26:31.617+0000] {logging_mixin.py:188} INFO - Error initializing GCS client: Your default credentials were not found. To set up Application Default Credentials, see https://cloud.google.com/docs/authentication/external/set-up-adc for more information.
[2025-04-06T15:26:31.618+0000] {logging_mixin.py:188} INFO - Deleted unneeded file: /tmp/CS50x-Coursework-Projects/Week 8 HTML, CSS, JavaScript/Homepage/README.md
[2025-04-06T15:26:31.621+0000] {_metadata.py:139} WARNING - Compute Engine Metadata server unavailable on attempt 1 of 3. Reason: [Errno 111] Connection refused
[2025-04-06T15:26:31.625+0000] {_metadata.py:139} WARNING - Compute Engine Metadata server unavailable on attempt 2 of 3. Reason: [Errno 111] Connection refused
[2025-04-06T15:26:31.631+0000] {_metadata.py:139} WARNING - Compute Engine Metadata server unavailable on attempt 3 of 3. Reason: [Errno 111] Connection refused
[2025-04-06T15:26:31.632+0000] {_default.py:338} WARNING - Authentication failed using Compute Engine authentication due to unavailable metadata server.
[2025-04-06T15:26:31.633+0000] {logging_mixin.py:188} INFO - Error initializing GCS client: Your default credentials were not found. To set up Application Default Credentials, see https://cloud.google.com/docs/authentication/external/set-up-adc for more information.
[2025-04-06T15:26:31.638+0000] {_metadata.py:139} WARNING - Compute Engine Metadata server unavailable on attempt 1 of 3. Reason: [Errno 111] Connection refused
[2025-04-06T15:26:31.642+0000] {_metadata.py:139} WARNING - Compute Engine Metadata server unavailable on attempt 2 of 3. Reason: [Errno 111] Connection refused
[2025-04-06T15:26:31.645+0000] {_metadata.py:139} WARNING - Compute Engine Metadata server unavailable on attempt 3 of 3. Reason: [Errno 111] Connection refused
[2025-04-06T15:26:31.647+0000] {_default.py:338} WARNING - Authentication failed using Compute Engine authentication due to unavailable metadata server.
[2025-04-06T15:26:31.648+0000] {logging_mixin.py:188} INFO - Error initializing GCS client: Your default credentials were not found. To set up Application Default Credentials, see https://cloud.google.com/docs/authentication/external/set-up-adc for more information.
[2025-04-06T15:26:31.651+0000] {_metadata.py:139} WARNING - Compute Engine Metadata server unavailable on attempt 1 of 3. Reason: [Errno 111] Connection refused
[2025-04-06T15:26:31.655+0000] {_metadata.py:139} WARNING - Compute Engine Metadata server unavailable on attempt 2 of 3. Reason: [Errno 111] Connection refused
[2025-04-06T15:26:31.658+0000] {_metadata.py:139} WARNING - Compute Engine Metadata server unavailable on attempt 3 of 3. Reason: [Errno 111] Connection refused
[2025-04-06T15:26:31.659+0000] {_default.py:338} WARNING - Authentication failed using Compute Engine authentication due to unavailable metadata server.
[2025-04-06T15:26:31.660+0000] {logging_mixin.py:188} INFO - Error initializing GCS client: Your default credentials were not found. To set up Application Default Credentials, see https://cloud.google.com/docs/authentication/external/set-up-adc for more information.
[2025-04-06T15:26:31.662+0000] {_metadata.py:139} WARNING - Compute Engine Metadata server unavailable on attempt 1 of 3. Reason: [Errno 111] Connection refused
[2025-04-06T15:26:31.665+0000] {_metadata.py:139} WARNING - Compute Engine Metadata server unavailable on attempt 2 of 3. Reason: [Errno 111] Connection refused
[2025-04-06T15:26:31.668+0000] {_metadata.py:139} WARNING - Compute Engine Metadata server unavailable on attempt 3 of 3. Reason: [Errno 111] Connection refused
[2025-04-06T15:26:31.669+0000] {_default.py:338} WARNING - Authentication failed using Compute Engine authentication due to unavailable metadata server.
[2025-04-06T15:26:31.670+0000] {logging_mixin.py:188} INFO - Error initializing GCS client: Your default credentials were not found. To set up Application Default Credentials, see https://cloud.google.com/docs/authentication/external/set-up-adc for more information.
[2025-04-06T15:26:31.671+0000] {logging_mixin.py:188} INFO - Deleted unneeded file: /tmp/CS50x-Coursework-Projects/Week 8 HTML, CSS, JavaScript/Homepage/images/gPigFace.png
[2025-04-06T15:26:31.672+0000] {logging_mixin.py:188} INFO - Deleted unneeded file: /tmp/CS50x-Coursework-Projects/Week 8 HTML, CSS, JavaScript/Homepage/images/GuineaPigMainPage.jpg
[2025-04-06T15:26:31.672+0000] {logging_mixin.py:188} INFO - Deleted unneeded file: /tmp/CS50x-Coursework-Projects/Week 8 HTML, CSS, JavaScript/Homepage/images/crap_cage.jpeg
[2025-04-06T15:26:31.673+0000] {logging_mixin.py:188} INFO - Deleted unneeded file: /tmp/CS50x-Coursework-Projects/Week 8 HTML, CSS, JavaScript/Homepage/images/studious_pig.jpg
[2025-04-06T15:26:31.673+0000] {logging_mixin.py:188} INFO - Deleted unneeded file: /tmp/CS50x-Coursework-Projects/Week 8 HTML, CSS, JavaScript/Homepage/images/great_cage.jpg
[2025-04-06T15:26:31.674+0000] {logging_mixin.py:188} INFO - Deleted unneeded file: /tmp/CS50x-Coursework-Projects/Week 8 HTML, CSS, JavaScript/Homepage/sounds/gpig_squeak.m4a
[2025-04-06T15:26:31.675+0000] {logging_mixin.py:188} INFO - Deleted unneeded file: /tmp/CS50x-Coursework-Projects/.git/HEAD
[2025-04-06T15:26:31.676+0000] {logging_mixin.py:188} INFO - Deleted unneeded file: /tmp/CS50x-Coursework-Projects/.git/index
[2025-04-06T15:26:31.676+0000] {logging_mixin.py:188} INFO - Deleted unneeded file: /tmp/CS50x-Coursework-Projects/.git/packed-refs
[2025-04-06T15:26:31.677+0000] {logging_mixin.py:188} INFO - Deleted unneeded file: /tmp/CS50x-Coursework-Projects/.git/description
[2025-04-06T15:26:31.678+0000] {logging_mixin.py:188} INFO - Deleted unneeded file: /tmp/CS50x-Coursework-Projects/.git/config
[2025-04-06T15:26:31.678+0000] {logging_mixin.py:188} WARNING - Processing CS50x-Coursework-Projects: 42it [00:00, 55.57it/s]
[2025-04-06T15:26:31.679+0000] {logging_mixin.py:188} INFO - Deleted unneeded file: /tmp/CS50x-Coursework-Projects/.git/refs/remotes/origin/HEAD
[2025-04-06T15:26:31.680+0000] {logging_mixin.py:188} INFO - Deleted unneeded file: /tmp/CS50x-Coursework-Projects/.git/refs/heads/main
[2025-04-06T15:26:31.680+0000] {logging_mixin.py:188} INFO - Deleted unneeded file: /tmp/CS50x-Coursework-Projects/.git/logs/HEAD
[2025-04-06T15:26:31.681+0000] {logging_mixin.py:188} INFO - Deleted unneeded file: /tmp/CS50x-Coursework-Projects/.git/logs/refs/remotes/origin/HEAD
[2025-04-06T15:26:31.682+0000] {logging_mixin.py:188} INFO - Deleted unneeded file: /tmp/CS50x-Coursework-Projects/.git/logs/refs/heads/main
[2025-04-06T15:26:31.682+0000] {logging_mixin.py:188} INFO - Deleted unneeded file: /tmp/CS50x-Coursework-Projects/.git/objects/pack/pack-625f2b09b07e6f10e75ea83a6781c6f9dba45a22.idx
[2025-04-06T15:26:31.683+0000] {logging_mixin.py:188} INFO - Deleted unneeded file: /tmp/CS50x-Coursework-Projects/.git/objects/pack/pack-625f2b09b07e6f10e75ea83a6781c6f9dba45a22.pack
[2025-04-06T15:26:31.684+0000] {logging_mixin.py:188} INFO - Deleted unneeded file: /tmp/CS50x-Coursework-Projects/.git/hooks/pre-receive.sample
[2025-04-06T15:26:31.684+0000] {logging_mixin.py:188} INFO - Deleted unneeded file: /tmp/CS50x-Coursework-Projects/.git/hooks/fsmonitor-watchman.sample
[2025-04-06T15:26:31.685+0000] {logging_mixin.py:188} INFO - Deleted unneeded file: /tmp/CS50x-Coursework-Projects/.git/hooks/push-to-checkout.sample
[2025-04-06T15:26:31.685+0000] {logging_mixin.py:188} INFO - Deleted unneeded file: /tmp/CS50x-Coursework-Projects/.git/hooks/commit-msg.sample
[2025-04-06T15:26:31.686+0000] {logging_mixin.py:188} INFO - Deleted unneeded file: /tmp/CS50x-Coursework-Projects/.git/hooks/pre-rebase.sample
[2025-04-06T15:26:31.686+0000] {logging_mixin.py:188} INFO - Deleted unneeded file: /tmp/CS50x-Coursework-Projects/.git/hooks/update.sample
[2025-04-06T15:26:31.687+0000] {logging_mixin.py:188} INFO - Deleted unneeded file: /tmp/CS50x-Coursework-Projects/.git/hooks/post-update.sample
[2025-04-06T15:26:31.687+0000] {logging_mixin.py:188} INFO - Deleted unneeded file: /tmp/CS50x-Coursework-Projects/.git/hooks/pre-merge-commit.sample
[2025-04-06T15:26:31.688+0000] {logging_mixin.py:188} INFO - Deleted unneeded file: /tmp/CS50x-Coursework-Projects/.git/hooks/prepare-commit-msg.sample
[2025-04-06T15:26:31.689+0000] {logging_mixin.py:188} INFO - Deleted unneeded file: /tmp/CS50x-Coursework-Projects/.git/hooks/pre-applypatch.sample
[2025-04-06T15:26:31.689+0000] {logging_mixin.py:188} INFO - Deleted unneeded file: /tmp/CS50x-Coursework-Projects/.git/hooks/applypatch-msg.sample
[2025-04-06T15:26:31.690+0000] {logging_mixin.py:188} INFO - Deleted unneeded file: /tmp/CS50x-Coursework-Projects/.git/hooks/pre-push.sample
[2025-04-06T15:26:31.690+0000] {logging_mixin.py:188} INFO - Deleted unneeded file: /tmp/CS50x-Coursework-Projects/.git/hooks/pre-commit.sample
[2025-04-06T15:26:31.691+0000] {logging_mixin.py:188} INFO - Deleted unneeded file: /tmp/CS50x-Coursework-Projects/.git/info/exclude
[2025-04-06T15:26:31.692+0000] {logging_mixin.py:188} INFO - Deleted unneeded file: /tmp/CS50x-Coursework-Projects/Week 0 Scratch/Scratch Project.sb3
[2025-04-06T15:26:31.693+0000] {logging_mixin.py:188} INFO - Deleted unneeded file: /tmp/CS50x-Coursework-Projects/Week 5 Data Structures/README.md
[2025-04-06T15:26:31.694+0000] {logging_mixin.py:188} INFO - Deleted unneeded file: /tmp/CS50x-Coursework-Projects/Week 5 Data Structures/Inheritance/README.md
[2025-04-06T15:26:31.694+0000] {logging_mixin.py:188} INFO - Deleted unneeded file: /tmp/CS50x-Coursework-Projects/Week 5 Data Structures/Inheritance/inheritance.c
[2025-04-06T15:26:31.695+0000] {logging_mixin.py:188} INFO - Deleted unneeded file: /tmp/CS50x-Coursework-Projects/Week 5 Data Structures/Speller/README.md
[2025-04-06T15:26:31.696+0000] {logging_mixin.py:188} INFO - Deleted unneeded file: /tmp/CS50x-Coursework-Projects/Week 5 Data Structures/Speller/speller.c
[2025-04-06T15:26:31.696+0000] {logging_mixin.py:188} INFO - Deleted unneeded file: /tmp/CS50x-Coursework-Projects/Week 5 Data Structures/Speller/dictionary.h
[2025-04-06T15:26:31.696+0000] {logging_mixin.py:188} INFO - Deleted unneeded file: /tmp/CS50x-Coursework-Projects/Week 5 Data Structures/Speller/speller
[2025-04-06T15:26:31.697+0000] {logging_mixin.py:188} INFO - Deleted unneeded file: /tmp/CS50x-Coursework-Projects/Week 5 Data Structures/Speller/dictionary.c
[2025-04-06T15:26:31.697+0000] {logging_mixin.py:188} INFO - Deleted unneeded file: /tmp/CS50x-Coursework-Projects/Week 1 C/README.md
[2025-04-06T15:26:31.698+0000] {logging_mixin.py:188} INFO - Deleted unneeded file: /tmp/CS50x-Coursework-Projects/Week 1 C/Mario/README.md
[2025-04-06T15:26:31.699+0000] {logging_mixin.py:188} INFO - Deleted unneeded file: /tmp/CS50x-Coursework-Projects/Week 1 C/Mario/less/mario
[2025-04-06T15:26:31.699+0000] {logging_mixin.py:188} INFO - Deleted unneeded file: /tmp/CS50x-Coursework-Projects/Week 1 C/Mario/less/mario.c
[2025-04-06T15:26:31.700+0000] {logging_mixin.py:188} INFO - Deleted unneeded file: /tmp/CS50x-Coursework-Projects/Week 1 C/Cash/README.md
[2025-04-06T15:26:31.701+0000] {logging_mixin.py:188} INFO - Deleted unneeded file: /tmp/CS50x-Coursework-Projects/Week 1 C/Cash/cash
[2025-04-06T15:26:31.701+0000] {logging_mixin.py:188} INFO - Deleted unneeded file: /tmp/CS50x-Coursework-Projects/Week 1 C/Cash/cash.c
[2025-04-06T15:26:31.702+0000] {logging_mixin.py:188} INFO - Deleted unneeded file: /tmp/CS50x-Coursework-Projects/Week 1 C/Hello, It’s Me/README.md
[2025-04-06T15:26:31.702+0000] {logging_mixin.py:188} INFO - Deleted unneeded file: /tmp/CS50x-Coursework-Projects/Week 1 C/Hello, It’s Me/hello.c
[2025-04-06T15:26:31.703+0000] {logging_mixin.py:188} INFO - Deleted unneeded file: /tmp/CS50x-Coursework-Projects/Week 1 C/Hello, It’s Me/hello
[2025-04-06T15:26:31.704+0000] {logging_mixin.py:188} INFO - Deleted unneeded file: /tmp/CS50x-Coursework-Projects/Week 3 Algorithms/README.md
[2025-04-06T15:26:31.705+0000] {logging_mixin.py:188} INFO - Deleted unneeded file: /tmp/CS50x-Coursework-Projects/Week 3 Algorithms/Sort/answers.txt
[2025-04-06T15:26:31.706+0000] {logging_mixin.py:188} INFO - Deleted unneeded file: /tmp/CS50x-Coursework-Projects/Week 3 Algorithms/Plurality/plurality.c
[2025-04-06T15:26:31.707+0000] {logging_mixin.py:188} INFO - Deleted unneeded file: /tmp/CS50x-Coursework-Projects/Week 3 Algorithms/Plurality/plurality
[2025-04-06T15:26:31.708+0000] {logging_mixin.py:188} INFO - Deleted unneeded file: /tmp/CS50x-Coursework-Projects/Week 3 Algorithms/Plurality/README (9).md
[2025-04-06T15:26:31.708+0000] {logging_mixin.py:188} INFO - Deleted unneeded file: /tmp/CS50x-Coursework-Projects/Week 3 Algorithms/Tideman/tideman
[2025-04-06T15:26:31.709+0000] {logging_mixin.py:188} INFO - Deleted unneeded file: /tmp/CS50x-Coursework-Projects/Week 3 Algorithms/Tideman/README.md
[2025-04-06T15:26:31.711+0000] {logging_mixin.py:188} INFO - Deleted unneeded file: /tmp/CS50x-Coursework-Projects/Week 3 Algorithms/Tideman/tideman.c
[2025-04-06T15:26:31.712+0000] {logging_mixin.py:188} WARNING - Processing CS50x-Coursework-Projects: 70it [00:00, 89.79it/s]
[2025-04-06T15:26:31.720+0000] {logging_mixin.py:188} INFO - Deleted temporary folder: /tmp/CS50x-Coursework-Projects
[2025-04-06T15:26:31.721+0000] {logging_mixin.py:188} INFO - Repository Bhasha-ai already cloned. Skipping.
[2025-04-06T15:26:31.722+0000] {logging_mixin.py:188} INFO - Data loading completed.
[2025-04-06T15:26:31.723+0000] {gcs_fileupload_dag.py:18} INFO - Done running data_loading.
[2025-04-06T15:26:31.724+0000] {python.py:201} INFO - Done. Returned value was: None
[2025-04-06T15:26:31.739+0000] {taskinstance.py:1138} INFO - Marking task as SUCCESS. dag_id=gcs_file_upload_dag, task_id=run_data_loading, execution_date=20250406T150000, start_date=20250406T152027, end_date=20250406T152631
[2025-04-06T15:26:31.793+0000] {local_task_job_runner.py:234} INFO - Task exited with return code 0
[2025-04-06T15:26:31.824+0000] {taskinstance.py:3280} INFO - 0 downstream tasks scheduled from follow-on schedule check
